%If you are presenting work which has been previously published, acknowledge this here.
% ***************************************************
% How to introduce a previously published chapter
% ***************************************************
%This is an example of how you might introduce a chapter that has been published previously. 
\cleartoevenpage
\pagestyle{empty}	
%Use this command (above) to suppress the header from the preceding chapter.

\noindent
The following publication has been incorporated as Chapter~\ref{Chap:phylo-signal}:\\

\noindent
\fullcite{macklin-cordes_phylogenetic_2020}

\begin{table}[h]
	\centering
	\begin{tabular}{clr}
		\toprule
		Contributor & Statement of contribution & \% \\
		\midrule
		\textbf{Jayden Macklin-Cordes}	& initial concept			& 50 \\
		                                & scripting, analysis       & 80 \\
		                                & writing of text 			& 90 \\
										& proof-reading				& 20 \\
										& preparation of figures 	& 75 \\
		\midrule
		Claire Bowern                   & supplying data            & 10 \\
										& supervision, guidance 	& 5  \\
										& proof-reading             & 10 \\
		\midrule
		Erich Round 					& initial concept			& 50 \\
		                                & supplying data            & 90 \\
		                                & scripting, analysis       & 20 \\
		                                & supervision, guidance 	& 95 \\
		                                & writing of text 			& 10 \\
										& proof-reading				& 70 \\
										& preparation of figures 	& 25 \\
										
		\bottomrule
	\end{tabular}
\end{table}

\noindent
Data for this paper comes from the Ausphon Lexicon database, created and maintained by Erich Round and extending on the CHIRILA database created and maintained by Claire Bowern. The Pama-Nyungan reference phylogeny was inferred and supplied by Claire Bowern.\\

\noindent
Responsibility for the study's initial conceptualisation and development is shared approximately equally between Macklin-Cordes and Round, representing the culmination of innumerable discussions over a long period of time.\\

\noindent
Experimental design, scripting and analysis was conducted by Macklin-Cordes (though note that the scripts depend considerably on functions developed by Round). Round contributed text to Section \ref{wordlists}. Macklin-Cordes drafted the remainder of the text. The 'swatch' visual device (e.g. Figure \ref{fig:k-swatch}) was created by Round and minimally adapted by Macklin-Cordes in this paper. Remaining figures were created by Macklin-Cordes with guidance from Round. Macklin-Cordes and Round revised text with feedback from three anonymous reviewers. Bowern assisted with proofreading.

% ***************************************************
% Example of an internal chapter
% ***************************************************
%This is an internal chapter of the thesis.
%If you have a long title, you can supply an abbreviated version to print in the Table of Contents using the optional argument to the \chapter command.
\chapter[Phylogenetic signal in phonotactics]{Phylogenetic signal in phonotactics}
\label{Chap:phylo-signal}	%CREATE YOUR OWN LABEL.
\pagestyle{headings}

% ********* Enter your text below this line: ********

Phylogenetic methods have broad potential in linguistics beyond tree inference. Here, we show how a phylogenetic approach opens the possibility of gaining historical insights from entirely new kinds of linguistic data---in this instance, statistical phonotactics. We extract phonotactic data from 111 Pama-Nyungan vocabularies and apply tests for \emph{phylogenetic signal}, quantifying the degree to which the data reflect phylogenetic history. We test three datasets: (1) binary variables recording the presence or absence of \emph{biphones} (two-segment sequences) in a lexicon (2) frequencies of transitions between segments, and (3) frequencies of transitions between natural sound classes. Australian languages have been characterised as having a high degree of phonotactic homogeneity. Nevertheless, we detect phylogenetic signal in all datasets. Phylogenetic signal is higher in finer-grained frequency data than in binary data, and highest in natural-class-based data. These results demonstrate the viability of employing a new source of readily extractable data in historical and comparative linguistics.

\hypertarget{phy-sig-intro}{%
\section{Introduction}\label{phy-sig-intro}}

A defining methodological development in 21st century historical linguistics has been the adoption of computational phylogenetic methods for inferring phylogenetic trees of languages \autocite{bowern_computational_2018}. The computational implementation of these methods means that it is possible to analyse large samples of languages, thereby inferring the phylogeny (evolutionary tree) of large language families at a scale and level of internal detail that would be difficult, if not impossible, to ascertain manually by a human researcher \autocite[p.~827]{bowern_computational_2012}. There is more to phylogenetics than building trees, and there exists untapped potential to explore the language sciences and human history with a phylogenetic approach. For example, in linguistics, phylogenetic methods have been integrated with geography to infer population movements \autocites{walker_bayesian_2011}{bouckaert_origin_2018}. In comparative biology, phylogenetic methods have been applied profitably to investigations of community ecology \autocite{webb_phylogenies_2002}, ecological niche conservatism \autocite{losos_phylogenetic_2008}, paeleobiology \autocite{sallan_heads_2012} and quantitative genetics \autocite{de_villemereuil_general_2014}. At the heart of these methods, however, is a sound understanding of the evolutionary dynamics of comparative structures. In this paper, we pesent a foundational step by detecting \emph{phylogenetic signal}, the tendency of related species (in our case, language varieties) to share greater-than-chance resemblances \autocite{blomberg_tempo_2002}, in quantitative phonotactic variation.

Throughout recent advances in linguistic phylogenetics, less attention has been paid to methodological development at the stage of data preparation. Large-scale linguistic phylogenetic studies continue, by-and-large, to rely on lexical data which have been manually coded according to the principles of the \emph{comparative method} \autocites[as described by][]{meillet_methode_1925}{campbell_historical_2004}{weiss_comparative_2014}---the comparative method being the long-standing gold-standard of historical linguistic methodology \autocites{chang_ancestry-constrained_2015}{bouckaert_origin_2018}{kolipakam_bayesian_2018}. This article demonstrates that phonotactics can also present a source of historical information. We find that, for a sample of 111 Pama-Nyungan language varieties, collections of relatively simple and semi-automatically-extracted phonotactic variables (termed \emph{characters} throughout) contain phylogenetic signal. This has positive implications for the utility of such phonotactic data in linguistic phylogenetic inquiry, but also introduces methodological considerations for phonological typology.

We discuss firstly the motivations for looking at phonotactics as a source of historical signal, and we give some broader scientific context that motivates the methodological approach we take later on. In Sections \ref{phy-sig-bin}--\ref{phy-sig-classes}, we present tests for phylogenetic signal in phonotactic characters extracted from wordlists for 111 Pama-Nyungan language varieties. Section \ref{materials} details the materials used and reference phylogeny. Section \ref{phy-sig-bin} tests for phylogenetic signal in binary characters that code the presence or absence of biphones (two-segment sequences) in each wordlist, capturing information on the permissibility of certain sequences in a language. Section \ref{phy-sig-cont} also tests for phylogenetic signal in biphones, but extracts a finer-grained level of variation by taking into account the relative frequencies of transitions between segments. Section \ref{phy-sig-classes} groups segments into natural sound classes and tests for phylogenetic signal in characters coding the relative frequencies of transitions between different classes. We conclude with discussion of the limitations of the study design, implications of the results and directions for future research.

\hypertarget{motiv}{%
\subsection{Motivations}\label{motiv}}

There are at least two reasons why consideration of alternative data sources could be fruitful in historical linguistics. The first is that a bottleneck persists in linguistic phylogenetics when it comes to data processing. The data for most linguistic phylogenetic studies are lexical cognate data---typically binary characters marking the presence or absence of a cognate word in the lexicon of each language---which have been assembled from the manual judgements of expert linguists using the traditional comparative method of historical linguistics \autocite[e.g.][]{weiss_comparative_2014}. Although data assembled in this way is likely to remain the gold-standard in historical linguistics for the foreseeable future, it nevertheless constitutes slow and painstaking work \autocites[notwithstanding efforts to automate parts of the process; see][]{list_potential_2017}{rama_are_2018}{list_sequence_2018}. This restricts the pool of languages that can be included in phylogenetic research to those that have been more thoroughly documented, introducing the risk of a sampling bias, where relatively well-studied regions of the global linguistic landscape are over-represented in historical and comparative work.

The second motivation for considering alternative historical data sources is that there are inherent limitations associated with lexical data. Undetected semantic shifts and borrowed lexical items erode patterns of vertical inheritance in a language's lexicon. Put another way, these changes create noise in the historical signal of a language's lexicon. Chance resemblances between non-historically cognate words are another source of noise in lexical data. Eventually, semantic shifts, borrowings and chance resemblances will accumulate to a point where genuine historical signal is indistinguishable from noise. This imposes a maximal cap on the time-depth to which the comparative method can be applied, which is typically assumed to sit somewhere around 10,000 years BP, based on the approximate age of the Afro-Asiatic family \autocite[p.~135]{nichols_sprung_1997}. Some phylogenetic studies have attempted to push back the time-depth limitations of lexical data by using characters that code for a range of grammatical features, under the rationale that a language's grammatical structures should be more historically stable than its lexicon \autocites{dunn_structural_2005}{rexova_cladistic_2006}. However, contrary to expectation, a recent study suggests that grammatical characters evolve faster than lexical data \autocite{greenhill_evolutionary_2017}. Differing rates of evolution are also found in phonology, specifically the rates of change in vowel inventories versus consonant inventories \autocites{moran_differential_2018}{moran_investigating_2020}. An additional issue with grammatical characters is that the space of possibilities for a grammatical variable is often restricted. This means that chance similarities due to \emph{homoplasy} (parallel historical changes) will be much more frequent \autocite[c.f.][]{chang_ancestry-constrained_2015}. For example, many unrelated languages will share the same basic word order by chance, because there is a logical limit on the number of basic word order categories.

\hypertarget{why-phonotactics}{%
\subsection{Phonotactics as a source of historical signal}\label{why-phonotactics}}

The motivation for considering a language's phonotactics as a potential source of historical information is based partly on practical and partly on theoretical observations. From a practical perspective, it is possible to extract phonotactic data with relative ease, at scale, from otherwise resource-poor languages. This is because the bulk of a language's phonotactic system can be extracted directly from phonemicized wordlists. As long as there is a wordlist of suitable length \autocite{dockum_swadesh_2019} and a phonological analysis of the language, phonotactic information can be deduced and coded from the sequences of segments found in the wordlist with a high degree of automation. This modest minimum requirement with regards to language resources is a valuable property in less documented linguistic regions of the world. We detail the process of data extraction for this study in Section \ref{materials} below.

An additional benefit of extracting phonotactic data from wordlists is the potential for expanding the depth of comparative datasets. Although macro-scale studies, including hundreds or even thousands of the world's languages (in other words, \emph{broader} datasets), are increasingly common in comparative linguistics, less attention has been paid to the number of characters per language (dataset \emph{depth}). It is quite a different situation in evolutionary biology, where there has been tremendous growth in whole genome sequencing, thanks to technological advances and falling costs \autocites{delsuc_phylogenomics_2005}{wortley_how_2005}. This, consequently, has led to tremendous growth in the depth of biological datasets. This is an important consideration because the quantity of characters required by modern computational phylogenetic methods can be substantial \autocites{wortley_how_2005}{marin_undersampling_2018}. Certainly, phonotactic data is unlikely to approach the scale of large genomic datasets in biology, but it could effectively deepen historical linguistic datasets.

From a theoretical perspective, there is reason to suspect that the phonotactics of a language preserve a degree of historical signal. There is some evidence that when a borrowed word enters the lexicon of a language, speakers tend to adapt it to suit the phonotactic patterns of that language \autocites{hyman_role_1970}{silverman_multiple_1992}{crawford_adaptation_2009}{kang_loanword_2011}. Consequently, in such a case the historical phonotactic structure of the lexicon remains largely intact, even as particular ancestral words are lost and replaced \autocite[a property termed \emph{pertinacity} by][]{dresher_main_2005}. Similarly, historical phonotactic properties of a language will remain in the phonotactics of a language in the case of an undetected semantic shift.

Laboratory evidence shows that speakers have a high degree of sensitivity to the statistical distribution of phonological segments and structures when producing novel words {[}for example, \textcite{coleman_stochastic_1997}; \textcite{albright_rules_2003}; \textcite{hayes_stochastic_2006}, among others \autocite[see][pp.~20--21]{gordon_phonological_2016}. Lexical innovation then, should have a relatively conservative impact on the frequency distributions of phonotactic characters. Every new word that enters a language's lexicon will have a minute impact on the frequencies of segments and particular sequences of segments in that language. But, over time, the cumulative effect of new lexicon entering a language on phonological and phonotactic frequency distributions will be more modest than if speakers generated new words with no regard for existing frequencies. Thus, there is reason to expect that quantitative phonotactic characters are likely to be conservative.

This is not to say that a language's phonotactic system remains completely immobile over time. Phonotactic systems are affected by sound changes and are not totally immune to borrowing. As mentioned above, frequencies of phonotactic characters will shift, however gradually, with the accumulation of lexical innovations. We make no strong claim about phonotactics being the key to a language's history. We merely note there are grounds to expect that phonotactic data will often be historically conservative, relative to cognate data which contains noise from lexical innovation, borrowing and semantic shift. Correspondingly, our hypothesis is that phonotactic data will contain relatively strong historical signal, which we test in Sections \ref{phy-sig-bin}--\ref{phy-sig-classes} below.

Many kinds of phonotactic structure exist, which could be studied phylogenetically. Here, because we wish to adhere to the basic methodological principle of studying maximally simple and clear cases first before progressing to more complex ones, we limit ourselves to the simplest of phonotactic structures, namely biphones. That being said, there is every reason to expect our results would generalise, perhaps with interesting variations, to other phonotactic structures. Moreover, many of those structures would have the same benefits as our biphones, in terms of their being readily generated in an automated fashion from wordlists. This will be a promising direction for future investigation.

\hypertarget{phylo-sig}{%
\section{Phylogenetic signal}\label{phylo-sig}}

The concept of \emph{phylogenetic signal} \autocites{blomberg_tempo_2002}[p.~717]{blomberg_testing_2003} originates in comparative biology, where it refers to the tendency of phylogenetically related species to resemble one another to a greater degree than would otherwise be expected by chance. This expectation derives from the evolutionary history shared between species. Two closely-related species, which share a relatively recent common ancestor, have had less time in which to diverge evolutionarily. We expect more distantly-related species, whose most recent common ancestor lies much further in the past, to tend to be more different, since they have spent longer on separate evolutionary paths.

Phylogenetic signal manifests itself as \emph{phylogenetic autocorrelation} in comparative studies. That is, species observations in a comparative dataset tend not to behave as independent data points, but rather pattern as a function of the amount of shared evolutionary history between species. For many statistical methods that assume data are independent and identically distributed (i.i.d.), this is a problem. Phylogenetic autocorrelation has long been recognised as an issue in linguistic typology and comparative biology, and both fields share comparable histories of developing sampling methodologies that attempt to correct for or offset phylogenetic relatedness in some way. More recent times have seen the rise of \emph{phylogenetic comparative methods}, statistical methods that directly account for phylogenetic autocorrelation, rather than offsetting it, beginning with foundational works by \textcite{felsenstein_phylogenies_1985} and \textcite{grafen_phylogenetic_1989}\footnote{See \textcite{nunn_comparative_2011} for discussion.}. Although now practically ubiquitous in comparative biology, uptake of phylogenetic comparative methods has been slower in comparative linguistics \autocites[notwithstanding studies such as][]{dunn_evolved_2011}{maurits_tracing_2014}{verkerk_diachronic_2014}{birchall_comparison_2015}{zhou_quantifying_2015}{calude_typology_2016}{dunn_dative_2017}{verkerk_phylogenetic_2017}{widmer_np_2017}{blasi_human_2019}.

Since the turn of the century, methods have been developed for explicitly quantifying the degree of phylogenetic signal in a dataset \autocite[p.~591]{revell_phylogenetic_2008}. Measuring phylogenetic signal can be the first step of a comparative study, to test whether there is sufficient phylogenetic signal to necessitate implementation of a phylogenetic comparative method in a later stage of analysis, or to establish the suitability of standard statistical methods if no phylogenetic signal is detected. Measures of phylogenetic signal can also be used to re-evaluate the validity of older results that pre-date modern phylogenetic comparative methods, as in \textcite{freckleton_phylogenetic_2002}. In other instances, the presence or absence of phylogenetic signal in certain data may be an interesting result in itself. In this study, we present a novel source of linguistic data which traditionally has not been considered a salient source of historical signal for historical linguistic study (indeed, given descriptions of Australian languages, it may have been considered a particularly unlikely source of historical signal; see Section \ref{sample}). We use measures of phylogenetic signal to test the hypothesis that our data contain historical information and, therefore, could contribute to future historical linguistic study.

\textcite{blomberg_testing_2003} provide a set of statistics for measuring phylogenetic signal, which remains prevalent today \autocites[for example,][]{balisi_dietary_2018}{hutchinson_contemporary_2018}{leff_predicting_2018}. We use one of these statistics, \(K\). The \(K\) statistic has the desirable property of being independent of the size and shape of the phylogenetic tree being investigated, which means that studies with different sample sizes can be compared directly. Briefly \autocite[following][p.~722]{blomberg_testing_2003}, the calculation of \(K\) requires three components: (i) character data (i.e., observations for the variable of interest); (ii) a \emph{reference phylogeny}, a phylogenetic tree which has been generated independently from the character data; and (iii) a \emph{Brownian motion} model of evolution.\footnote{A \emph{Brownian motion} model of evolution describes a model of character evolution where the character can move up or down with equal probability as it evolves through time. Under this model of evolution, variance in character values throughout a phylogeny will increase proportionally as time elapses.} These components entail two assumptions of the method: the assumption that the reference phylogeny is an accurate representation of the phylogenetic history of the populations being studied and the assumption that Brownian motion accurately models the evolution of the character data. In practice, the reference phylogeny will be subject to uncertainty. We return to this point in Section \ref{discussion} and evaluate the robustness of our results against phylogenetic uncertainty. Similarly, in practice, the Brownian motion model may not be realistic. Nevertheless, it is a simple model and straightforward to implement, and thus commonly used as a starting point before exploring more complex models of evolution later on. We discuss this further in Section \ref{discussion} and outline possible extensions to the model for future study, taking sound change processes into account. To the extent that Brownian motion fails to model the evolution of phonotactic characters, this should make it more difficult to detect phylogenetic signal.

The \(K\) statistic is then calculated by, firstly, taking the mean squared error of the data (\(MSE_0\)), as measured from a \emph{phylogenetic mean}\footnote{Simply taking the mean of some variable would be misleading in cases where members of a particularly large clade happen to share similar values at an extreme end of the range. A \emph{phylogenetic mean} is an estimate of the mean which takes into account any overrepresentation by larger subclades \autocite[see, for example,][]{garland_jr._polytomies_1999}.}, and dividing it by the mean squared error of the data (\(MSE\)), calculated using a variance-covariance matrix of phylogenetic distances between tips in the reference tree \autocite[see][ for a complete formula]{blomberg_testing_2003}. This latter value, \(MSE\), will be small when the pattern of covariance in the data matches what would be expected given the phylogenetic distances in the reference tree, leading to a high \(MSE_0/MSE\) ratio and vice versa. Thus, a high \(MSE_0/MSE\) ratio indicates higher phylogenetic signal. Finally, the observed \(MSE_0/MSE\) ratio can be scaled according to the expected \(MSE_0/MSE\) ratio given a Brownian motion model of evolution. This gives a statistic, \(K\), which can be compared directly between studies using different trees. When \(K = 1\), this suggests a perfect match between the covariance observed in the data and what would be expected given the reference tree and the assumption of Brownian motion evolution. When \(K < 1\), close relatives in the tree bear less resemblance in the data than would be expected under the Brownian motion assumption. \(K > 1\) is also possible---this occurs where there is less variance in the data than expected, given the Brownian motion assumption and divergence times suggested by the reference tree. In other words, relatives bear closer resemblance than would be expected if the variable evolved along the tree following a Brownian motion model of evolution.

\textcite{blomberg_testing_2003} also present a \emph{randomisation procedure} for testing whether the degree of phylogenetic signal in a dataset is statistically significant. The randomisation procedure utilises Felsenstein's (1985) \emph{phylogenetic independent contrasts} (PICs) method. Felsenstein's insight is that, although two character values (\(x\) and \(y\)) from two sister taxa cannot be considered independent due to phylogenetic autocorrelation, the contrast between them (\(x - y\)) is phylogenetically independent, since these values can only diverge in the time since the two sisters split from their most recent common ancestor. Given a set of character data and a phylogenetic tree, \textcite{felsenstein_phylogenies_1985} presents a method for harvesting a whole set of phylogenetically independent data points, PICs, which can be used for statistical analysis in lieu of the raw set of observations. \textcite{blomberg_testing_2003} take advantage of the expectation that, given a Brownian motion model of evolution, PIC variance is expected to be proportional to time. PICs among more closely-related taxa will tend to be lower than more distant relatives, since they have had less time to diverge from common ancestors. The randomisation procedure first extracts PICs for a given character and records the variance. Then, it extracts PICs and records the variance after randomly shuffling character data among taxa (thereby destroying phylogenetic signal). PIC variance is recorded typically for many thousands of such random permutations. If the true PIC variance (for original, unshuffled data) is lower than the variance of PICs in \textgreater{} 95\% of random permutations, the null hypothesis of no phylogenetic signal can be rejected at the conventional 95\% confidence level.

In this study, we also use a second statistic, \(D\), which was developed to measure phylogenetic signal in binary data. The \(D\) statistic is described by \textcite{fritz_selectivity_2010}. To summarise briefly, the \(D\) statistic is based on the sum of differences between sister tips and sister clades, \(\Sigma d\). First, differences between values at the tips of the tree are summed. Since \(D\) concerns binary variables, each taxon will either have a 0 or 1 value. At the level of the tips, then, all sister tips will either share the same value (in which case, the difference = 0) or one tip will have a 0 value and the other will have a 1 value (in which case, the difference = 1). Nodes immediately above the tree tips are given the average value of their daughter tips below (which, in a fully bifurcating phylogeny, will either be 0, 0.5 or 1). This process is repeated for all nodes in the tree, until a total sum of differences, \(\Sigma d\), is reached. At two extremes, data may be maximally clumped, such that all 1s are grouped together in the same clade in the tree and likewise for all 0s, or data may be maximally dispersed, such that no two sister tips share the same value (every pair of sisters contains a 1 and a 0, leading to a maximal sum of differences). Lying somewhere in between will be both (i) a distribution that is entirely random relative to phylogenetic structure and (ii) a distribution that is clumped exactly to the degree expected if the character evolved along the tree following a Brownian motion model of evolution. Two permutation procedures are used to determine where these two points lie for a given dataset and phylogenetic tree. Firstly, like Blomberg \emph{et al.}'s permutation test described above, character values are shuffled at random among tips of the tree many times over, thereby destroying phylogenetic signal. The sums of differences are taken from each random permutation to obtain a distribution of sums of differences, given phylogenetic randomness: \(\Sigma d_r\). Then, to obtain a contrasting distribution of sums of differences, the process of character evolution along the tree following a Brownian motion model is simulated many times over. Since Brownian motion is a model of evolution of continuous characters, and what we need here is a distribution of binary character values, and then binarises their values to 0 or 1 at the tips of the tree by observing whether they fall above or below a threshold value. This threshold is set to whatever level will produce the same proportion of 1s and 0s as observed in the real data. The sums of differences are then taken from each simulation, giving a distribution where phylogenetic signal is present: \(\Sigma d_b\). Finally, the \(D\) statistic is determined by scaling the observed sum of differences relative to the means of the two reference distributions just described:

\begin{equation}
D = \frac{\Sigma d_{obs} - mean\left( \Sigma d_{b} \right)}{mean\left( \Sigma d_{r} \right) - mean\left( \Sigma d_{b} \right)}
\end{equation}

Scaling \(D\) in this way provides a standardised statistic with the desirable property that it can be compared between different sets of data, with trees of different sizes and shapes, as with \(K\) for continuous characters. One disadvantage of \(D\), however, is that it requires quite large sample sizes (\textgreater{}50), below which it loses statistical power.

Two \(p\) values determine the statistical significance of \(D\), one each for the null hypotheses that \(D = 0\) (phylogenetic signal present) and \(D = 1\) (the character is distributed randomly relative to phylogenetic structure). These \(p\) values are obtained by comparing the observed \(D\) score to the two distributions of simulated \(D\) scores described above (\(\Sigma d_r\) and \(\Sigma d_b\)). The fraction of randomly simulated \(D\) scores smaller than observed \(D\) is taken as the \(p\) value for \(H_{0(D=1)}\). Likewise, the proportion of the simulated \(D\) scores greater than the observed \(D\) value is the \(p\) value for \(H_{0(D=0)}\).

\hypertarget{materials}{%
\section{Materials}\label{materials}}

Our study measures phylogenetic signal in a variety of types of phonotactic characters, extracted using semi-automated methods from wordlists within the Pama-Nyungan family (Australia). Throughout, we take the \emph{doculect} to be our unit of study. A doculect is a language variety as documented in a given resource \autocites{cysouw_towards_2007}{good_languoid_2013}. That is to say, we treat each wordlist as its own unit of study, without making any claims about the status of the documented language variety's status as a language or dialect. This agnosticism is advantageous in phylogenetic studies, since the terms `language' and `dialect' imply something about the relationship of a documented language variety to other documented language varieties, and a commitment to one term or the other therefore represents a phylogenetic assumption.

\hypertarget{sample}{%
\subsection{Language sample}\label{sample}}

Pama-Nyungan is by far the largest language family on the Australian continent, covering nearly 90\% of its landmass (everywhere except for three areas: part of the Top End, part of the Kimberley, and the whole of Tasmania) and encompassing around two-thirds of the languages present at the time of European settlement \autocite[p.~817]{bowern_computational_2012}. Pama-Nyungan was first proposed and named by Kenneth Hale \autocite[p.~136]{wurm_aboriginal_1963} and it has been the subject of considerable historical linguistic study since this time. Although the family has presented some challenges for historical linguistics, the phylogenetic unity of Pama-Nyungan has been established on traditional historical linguistic grounds \autocite{alpher_pama-nyungan:_2004} with many subgroups identified within \autocites[for example,][]{ogrady_languages_1966}{wurm_languages_1972}{austin_proto-kanyara_1981}. For an overview of the history of Pama-Nyungan classification, see \textcite[ch.~1--5]{bowern_australian_2004} and \textcite{koch_historical_2014}. \textcite{bowern_computational_2012} perform a computational phylogenetic analysis of Pama-Nyungan using lexical data from 194 language varieties, providing for the first time a fully bifurcating phylogeny of the entire Pama-Nyungan family. \textcite{bouckaert_origin_2018} subsequently perform a phylogeographic analysis using the same dataset, but refined and expanded to 306 language varieties and including a geographic element to estimate the point of origin and spread pattern of the family through time and space.

The Pama-Nyungan family provides an excellent test case for this study. It holds practical advantages which make the task of phonological comparison easier, but it also provides us with a deliberately high bar to clear from a theoretical perspective. Both of these features are a result of the unusual degree of phonological homogeneity observed among Australian languages. Australian languages have been noted for a degree of similarity between phonological inventories of contrastive segments that is exceptional and unexpected in light of the phylogenetic and geographical breadth of the family, the level of diversity observed in vocabulary and aspects of grammar, and the level of phonological diversity found in comparably-sized families of languages elsewhere in the world. This has been noted as early as \textcite{schmidt_gliederung_1919} and in more recent times by \textcite{capell_new_1956}, \textcite{voegelin_obtaining_1963}, \textcite{dixon_languages_1980}, \textcite{busby_distribution_1982}, \textcite{hamilton_phonetic_1996}, \textcite{baker_word_2014}, \textcite{bowern_standard_2017} and \textcite{round_segment_2020}, among others. This curious level of homogeneity extends to phonotactics too \autocites{dixon_languages_1980}{hamilton_phonetic_1996}{baker_word_2014}{round_phonotactics_2020}.

On one hand, the abundance of similar phonological inventories makes the task of comparison between them easier, because it limits the problem of \emph{dataset sparsity}. Consider a character coding the frequency of some sequence of two segments \(xy\) in a language: This character can only be compared between languages that contain both \(x\) and \(y\) segments in their inventories. If a language lacks either segment in its inventory, then the character will be coded as absent or missing (as distinct from 0, where a language possesses both segments but never permits them in sequence). We expect fewer missing values in Australia, where languages tend to share a large proportion of equivalent segments, when compared to other parts of the world where we would expect to see many more missing values.

On the other hand, an ostensibly high degree of phonological homogeneity, in spite of considerable phylogenetic diversity, presents challenges for historical linguistics. \textcite[p.~141]{baker_word_2014} and \textcite[p.~103]{alpher_pama-nyungan:_2004} have both written on the difficulties for historical reconstruction in Australia because of this. Moreover, a phylogeny implies some degree of historical divergence, but in the case of Australian languages, there would appear to be little by way of phonological divergence, let alone divergences which are phylogenetically patterned. We therefore choose to study an Australian language family as a deliberately difficult test case, where we expect the bar be set high with respect to detecting phylogenetic signal.

\textcite{gasser_revisiting_2014} counter prevailing views on Australian phonological homogeneity. They find that common assumptions, of the kind discussed above and commonly found repeated in reference grammars, mask a degree of variation which is otherwise revealed by, firstly, extracting data on segmental inventories directly from wordlists and, secondly, considering segmental frequencies extracted from wordlists. This result motivates our current approach; here, we are also concerned with matters of frequency, extracted directly from language wordlists. However, we look at different kinds of characters, pertaining not to single segments but to phonotactics, and consider them with respect to their phylogenetic implications.

\hypertarget{wordlists}{%
\subsection{Wordlists}\label{wordlists}}

Our Pama-Nyungan phonotactic data is extracted from 111 wordlists which are part of a database under development by the last author \autocite{round_ausphon-lexicon_2017}, extending the Chirila resources for Australian languages \autocite{bowern_chirila:_2016}. In this study we restrict our attention to the most accurate sources available, and use only lexical data that is compiled by trained linguists and for which the underlying dataset is available in published or archived form. Additionally, we restrict our sample to wordlists containing a minimum of 250 words. We include this cut-off since measurement accuracy is a concern for smaller wordlists. A documented wordlist is necessarily only a subset of the complete lexicon of a language and it is unclear how big a wordlist must be before frequency statistics begin to stabilise around a sufficient level of accuracy. There is some work in this space concerning frequences of single segments \autocite{dockum_swadesh_2019}, suggesting a rapid decline in the accuracy of phoneme frequencies as wordlists drop below 250 words. Longer wordlists will always be better, however we select 250 words as a reasonable compromise which maintains a generally broad coverage of Pama-Nyungan languages. We return to the subject of wordlist sizes and potential implications for our results in Section \ref{overall-robustness}

Bibliographic details for all underlying data is available in Section S2 of the Supplementary Information. Owing to differences in the length of primary sources, there is considerable diversity in the size of the lexicons we use. As shown in Figure \ref{fig:lex-size}, the difference from smallest to largest is over an order of magnitude (min. 250, max. 4955), with the middle fifty percent between 509 and 1367 items. Mean lexicon size is 1112 (\(SD\) 916))

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{fig/wordlist_sizes} 

}

\caption{Lexicon sizes.}\label{fig:lex-size}
\end{figure}

Original source data, which is typically orthographic and, if digital, is sometimes mixed with metadata or other extraneous material, has undergone extensive data scrubbing, conversion to phonemic form using language-specific orthography profiles \autocite{moran_unicode_2018}, and additional automated and manual error checking. These procedures ensure basic data cleanliness. Separately however, it has long been recognised that the segmental-phonological analysis of languages is a non-deterministic process \autocites{chao_non-uniqueness_1934}{hockett_problem_1963}{hyman_universals_2008}{dresher_contrastive_2009}. Two linguists faced with the same data may produce different analyses, not due to error but due to different applications of the very many analytic criteria that figure into any analysis of segments. Consequently, the cross-linguistic phonological record varies not only according to language facts per se, but due also to variation in the practice of linguistic analysis. Recent literature \autocites{lass_vowel_1984}{hyman_universals_2008}{van_der_hulst_phonological_2017}{round_matthew_2017}{kiparsky_formal_2018} emphasises the value of normalizing source descriptions prior to the analysis of cross-linguistic phonological datasets. This is not an information destroying process---it does not `standardise' languages---but it may shift information from one part of the representation (e.g., contrast between individual symbols) to another (e.g., contrasts between sequences of symbols), in order that information is located in a comparable way across the languages in the dataset, and therefore is more amenable to comparative analysis. Our wordlist data is normalised is this sense. Complex segments are split into simple sequences (e.g., prenasalised stops are split into a homorganic nasal + stop sequence); long vowels are represented as a sequence of identical short vowels, and vowel-glide-vowel sequences in which the glide is homorganic with either vowel are normalised to vowel-vowel; fortis consonants are represented as a sequence of identical short consonants, and positionally neutralised fortis/lenis stops as singletons; laminal consonants which do not figure in a pre-palatal versus dental opposition are represented as palatal, and rhotic glides which do not figure in an alveolar versus post-alveolar opposition are represented as post-alveolar \autocites[see also][]{round_phonemic_2019}{round_australian_2019}. The various phonotactic character sets were extracted from these normalised, comparably segmented wordlists.

\hypertarget{ref-phylogeny}{%
\subsection{Reference phylogeny}\label{ref-phylogeny}}

The reference phylogeny we use is a maximum clade credibility tree\footnote{Bayesian phylogenetic methods return not a single phylogenetic tree but a posterior distribution of many possible trees. These trees can be summarised into a single maximum clade credibility tree with confidence levels for each node in the tree, pertaining to how frequently that node appears in the posterior sample. It is the maximum clade credibility tree that we use for a reference phylogeny in this study. See \textcite{bowern_computational_2012} and \textcite{bouckaert_origin_2018} for a full explanation of the methods used to infer the phylogenies considered in this section.} of 285 Pama-Nyungan language varieties inferred using lexical cognate characters by the second author (Figure S1, Supplementary Information). It was inferred independently of this study, prior to this study's conception and without the involvement of the first and third authors. It was inferred using the same Stochastic Dollo model as \textcite{bowern_computational_2012}, but with an expanded and refined dataset. Further details of the model and phylogeny construction are described in \textcite{bowern_computational_2012}, \textcite{bowern_pama-nyungan_2015} and \textcite{bouckaert_origin_2018}. The cognate data used to infer the reference phylogeny is available on Zenodo \autocite{bowern_pama-nyungan_2018}. See Section S1 of the Supplementary Information for more information on the reference phylogeny.

We considered a reference tree from a newer phylogeographic analysis of Pama-Nyungan based on largely the same data plus further expansion to 304 doculects and continued refinement \autocite{bouckaert_origin_2018}, however, we opted against its use for this particular study. The reason for this is that, although Bayesian inference of phylogenetic tree topology is considered generally robust to the levels of lexical borrowing observed among Pama-Nyungan languages \autocites{greenhill_does_2009}{bowern_does_2011}, borrowing still has the effect of reducing branch lengths across the tree \autocite{greenhill_does_2009}. This effect, and consequently the accuracy of branch length estimates, is equally applicable to both trees considered here. However, the geographic element in the phylogeographic study uses, in part, branch lengths to model geographic dispersal. The posterior distribution of trees, which is jointly informed by cognate data and geography, may therefore show a bias towards geographically proximal languages whose apparent divergence times have been reduced by high rates of borrowing. Thus, although branch length estimates will be impacted by borrowing in any phylogenetic study of Pama-Nyungan, there is more chance of borrowing affecting topology in the phylogeographic study.

We consider it unlikely that the overall conclusions of the study would be altered by the choice of which version of the Pama-Nyungan phylogeny we use as a reference tree. Each of the studies referenced above produced highly congruent Pama-Nyungan phylogenies \autocite[see][ for a detailed comparison]{bouckaert_origin_2018}. Furthermore, \textcite{bouckaert_origin_2018} features fixed clade priors based on subgroups identified in earlier studies, so topological differences are constrained to some extent by design. Nevertheless, the accuracy of the reference tree is a key assumption of the methods we use in this study, and thus phylogenetic uncertainty is an important consideration. We return to this point in Section \ref{overall-robustness} and evaluate the overall robustness of our results to phylogenetic uncertainty by replicating a subset of phylognetic signal testing over a posterior sample of trees.

As discussed above, we treat each wordlist in our study as its own doculect. The reference tree in this study was inferred using a similar approach, while remaining less-commital about the particular status of the unit of analysis. Resources were sometimes combined for a particular language, but they are also frequently broken up into separate units, particularly when the resources come from different authors and different time periods. We have taken care to match the wordlists in this study to their exact or best corresponding tip in the reference tree. In most cases, the wordlists we use here are the same as those used to infer the reference phylogeny. In other cases, we use a different source to the one used in the reference phylogeny but there is, nevertheless, a straightforward one-to-one mapping between the language variety our wordlist represents and a corresponding tip in the tree. In one case, our Mudburra source \autocite{nash_mudburra_1988} matches neither of the sources for the two Mudburra tips in the reference phylogeny. However, the two varieties in the reference phylogeny have the same date. This entails that when either of them is removed from the tree, the exact same result is obtained in terms of tree geometry, which is what is significant for our investigation. Accordingly, we remove one and match our source to the other.

\hypertarget{phy-sig-bin}{%
\section{Phylogenetic signal in binary phonotactic data}\label{phy-sig-bin}}

In the simplest case, the phonotactics of different languages may be compared in terms of which sequences of two segments (\emph{biphones}) they permit and which they do not. If claims about the relative homogeneity of phonotactic constraints in Australian languages holds, then we would expect this kind of comparison to yield little, if any, phylogenetic signal.

In this test, we construct the dataset as follows: We automatically extract from all wordlists every unique sequence of two segments---or more accurately, sequences of \(xy\) where each of \(x\) and \(y\) is either a phonological segment or a word boundary `\#'. Each sequence becomes a character (variable) in the dataset, for which every language receives a binary value: 1 if the sequence \(xy\) is found in the language's wordlist (even if only once); 0 if the language contains both segments in its inventory but the sequence \(xy\) never appears in its wordlist; or, NA (not applicable, missing) if the language does not contain one (or both) of either segment \(x\) or \(y\) in its inventory (and therefore \emph{a prioi} cannot contain the sequence \(xy\)). Binary data of this kind represents sequence permissibility: Where a language contains both segments in its inventory, it will either permit them to appear together in sequence or it will not. In this respect, the information encoded by these characters is similar to what one might find in the phonotactics section of a descriptive grammar, where one often encounters a description in prose and/or a basic tabulation of which segments are permitted and where, within syllable and word structures. However, this kind of information is also, in a sense, quite coarse-grained, since there are only two possible values. A sequence which is very common in one language will be coded in exactly the same way as a sequence which only appears a handful of times in another language.

We apply the \(D\) test individually to each character in the dataset that meets two conditions: at least 50 non-missing values (due to the aforementioned reliability issue with sample sizes smaller than this) and at least one instance of variation (we do not test characters where all languages share identical 1 or 0 values). Given the extensive history of description of Australian languages as phonotactically homogenous, our prior expectation is that testing binary data will fail to yield significant phylogenetic signal. Indeed, we might expect that \(D\) will fall significantly below 0, indicating that values are clumped among tips on the reference phylogeny even more conservatively than would be expected if they had evolved in the same phylogenetic pattern as lexical data.

To evaluate the statistical significance of \(D\) for any given character, a \(p\) value is estimated for each of two null hypotheses: The null hypothesis that \(D = 1\) (\(H_{0(D=1)}\), character values are distributed randomly with regards to phylogeny) and the null hypothesis that \(D = 0\) (\(H_{0(D=0)}\), character values are distributed as could be expected if the character has evolved along the phylogeny according to a Brownian motion model). Each \(p\) value is calculated using a randomization procedure: A random distribution of \(D\) scores is acquired by randomly shuffling character values among tips on the tree for 10,000 permutations. The conventional cutoff for statistical significance is \(p = 0.05\). Here, we use the corresponding Bonferroni-corrected cutoff of 0.025.\footnote{Bonferroni correction is used because the conventional threshold for statistical significance, 0.05, represents the expected chance of a false discovery (false positive). This figure is known as the type I error rate (\(\alpha\)). The chance of a false discovery is multiplied when multiple tests are carried out. Bonferroni correction, which involves dividing the threshold for statistical significance by the number of tests being conducted, ensures that the chance of observing a false positive in any of the set of tests remains at the conventional rate, \(\alpha = 0.05\). In our case, two null hypotheses are tested for each character, hence we divide the threshold for statistical significance by two, ensuring the chance of a false positive for any particular character is 0.05.} For any given character, there are six possible results:
- \(D\) is significantly below 0. Character values are even more tightly clumped among sisters than Brownian motion alone would lead us to expect.
- \(D\) is significantly below 1 and not significantly different from 0. The data patterns phylogenetically, i.e., there is phylogenetic signal.
- \(D\) is significantly above 0 and below 1. The data is neither clearly random nor clearly phylogenetic.
- \(D\) is significantly above 0 and not significantly different from 1. It is consistent with randomness, not phylogeny.
- \(D\) is significantly above 1. It is even more dispersed than expected via a random process.
- \(D\) is not significantly distinct from 0 nor 1. The patterning of the data is indeterminate, and cannot be distinguished from randomness nor from Brownian phylogenetic evolution.

To summarise, the testing procedure proceeds as follows. For each binary biphone character:
- If the character has at least 50 non-NA values, then:
- If the character has at least one `1' and one `0' value (i.e.~not every value is identical), then:
- Calculate \(D\), and:
- Conduct randomisation procedure to calculate \(p\) for \(H_0: D = 0\), and:
- Conduct randomisation procedure to calculate \(p\) for \(H_0: D = 1\), and:
- A result is interpreted from the combination of \(D\) and two \(p\) values.

\hypertarget{phy-sig-bin-results}{%
\subsection{Results for binary phonotactic data}\label{phy-sig-bin-results}}

We estimate \(D\) for 415 biphone characters using a script based on the \emph{phylo.d} function in the \emph{caper} package \autocite{orme_caper:_2013}, implemented in the statistical software \emph{R} \autocite{r_core_team_r:_2017}\footnote{The dataset for this and subsequent tests are available on Zenodo at \url{http://doi.org/10.5281/zenodo.3610089}. This repository also includes a full table of results and the R scripts used to perform the analysis and produce figures for the paper. See Section S3 of the Supplementary Information for usage instructions and a full description of these materials.}. As described in Section \ref{phylo-sig}, \(D\) is calibrated by simulating character evolution under two models---one where the character evolves at random relative to phylogeny and one where the character evolves following a Brownian motion threshold model. In this study, we conduct 10,000 permutations of each model for each character. The 415 \(D\) values cluster centrally around a mean of \texttt{mean(d\_test\_table\$D)}. The distribution is leptokurtic (kurtosis = \texttt{kurtosis(d\_test\_table\$D)}), meaning there are more outliers relative to a normal distribution, making the distribution appear as a tall, narrow peak with long tails (Figure \ref{fig:d-density}). The standard deviation is large (\texttt{sd(d\_test\_table\$D)}).

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/d-density} 

}

\caption{Density of $D$ estimates for binary biphone characters. Dotted lines mark $D=0$, the phylogenetic expectation, and $D=1$, the random expectation. Mean $D$ for all characters is 0.43, marked in red.}\label{fig:d-density}
\end{figure}

The \(D\) test was of indeterminate significance for half of all characters (238 characters, 57\% of the dataset). The \(D\) scores for 157 characters (38\% of the total dataset) show evidence of phylogenetic signal. Just 16 characters (4\%) show the opposite result, where the character is consistent with randomness and there is no phylogenetic signal present. Both null hypotheses are rejected for the remaining 4 characters. Of these, 3 are more clumped than their phylogenetic expectation and \texttt{sum(d\_test\_table\ ==\ "0\ \textless{}\ D\ \textless{}\ 1\ (both\ H0s\ rejected)")} fall somewhere between phylogenetic and random expectations (\(0 < D < 1\)). None are more dispersed than the random expectation. The distribution of these results among different biphone characters is plotted in Figure \ref{fig:d-swatch}. Note that we expect around 5\% of null hypothesis rejections (approximately 18 of 180 rejected null hypotheses) to be false discoveries. Nevertheless, when considering the whole dataset as an ensemble rather than each character individually, a general result can be discerned. The clearest conclusion is that binary, permissibility-based characters tend to be low yielding in information, giving a statistically significant outcome in fewer than half of cases. Nevertheless, where a significant result can be determined, phylogenetic signal does tend to be present---to a degree that is perhaps surprising in light of previous literature describing the relative homogeneity of Australian phonotactic restrictions and their lack of utility in historical endeavours. This result suggests there may be a greater degree of historical information contained in Pama-Nyungan phonotactics than previously thought. However, it may be that a finer-grained approach to data extraction is needed in order to detect it.

These results can be compared to two earlier studies performing the same test on much smaller samples of languages. In the first, \textcite{macklin-cordes_high-definition_2015} find no evidence for phylogenetic signal in the Yolngu subgroup of Pama-Nyungan---rather, data are significantly over-clumped, suggesting a higher degree of conservatism in phonotactic restrictions relative to lexical data. They fail to reject a null hypothesis of \(D = 0\) for Ngumpin-Yapa, suggesting there may be a degree of phylogenetic signal in the Ngumpin-Yapa dataset. However, the pilot study results should be treated with caution---particularly the failure to reject the \(D = 0\) null hypothesis in the case of Ngumpin-Yapa---due to the small sample sizes (10 languages for Ngumpin-Yapa, 7 for Yolngu), well below the minimum of 50 taxa recommended by \textcite{fritz_selectivity_2010}. \textcite{dockum_phylogeny_2018} performs the same analysis using biphone characters from 20 Tai lects of the Kra-Dai family. In contrast to \textcite{macklin-cordes_high-definition_2015}, Dockum finds some evidence of phylogenetic signal in the Tai data and suggests perhaps the earlier result was due to insufficient variation in that particular language sample rather than a limitation of binary biphone characters per se. Although the low information yield from binary data is to be expected, our results here appear to support Dockum's conclusion.

\begin{figure}

{\centering \includegraphics[width=0.9\linewidth]{fig/d-sig-swatch} 

}

\caption{Phylogenetic signal significance testing for binary biphone characters. This grid colour-codes each biphone character according to the results of its respective significance tests. The grid is arranged such that the vertical axis represents the first segment of the biphone and the horizontal axis represents the second segment. Besides a tendency for phonotactic restrictions at word boundaries to show phylogenetic signal, few patterns stand out.}\label{fig:d-swatch}
\end{figure}

\hypertarget{phy-sig-bin-robustness}{%
\subsection{Robustness checks}\label{phy-sig-bin-robustness}}

Given a sufficient number of taxa for which data are available (\textgreater{}50), \(D\) scores should reflect a degree of phylogenetic signal present in the data, independently of tree size (the number of taxa) and shape (branching patterns). To check this, in Figure \ref{fig:d-scatterplots}(a) for each character we plot its \(D\) score against the number of doculects for which it had non-missing values. Irrespective of the number of doculects that supply non-missing values, the \(D\) scores appear to cluster centrally around mean \(D\) for the dataset, suggesting that \(D\) is not being unduly affected by missing values for particular characters. A second check leads to rather different results, however. We check whether skewed distributions of character values affects \(D\) scores (Figure \ref{fig:d-scatterplots}(b)). Here, we consider the distribution of 1s and 0s for each character and plot \(D\) against how skewed the distribution of character values is towards a particular value. For example, a character where there are 107 `1' values in the dataset and only 4 `0' values will have a skewing rating of 0.963964 (the count of `1' values, 107, divided by 111 total observations). Here, we find that when the ratio of 1s and 0s for a character is highly unequal, estimates of \(D\) tend towards extreme magnitudes while also being unrevealing, that is, statistically distinguishable neither from 0 nor 1.\footnote{Note that it is a desirable feature of the \(D\) test that it should return a lack of significance when there is a near-complete lack of variability in the data for the test to evaluate. This is an issue with the data, not the test.} As described above, Australian languages are known for homogeneity in phonological inventories and phonotactic restrictions, so consequently there are many characters with skewed distributions affecting the results. In Figures \ref{fig:d-density-filtered}--\ref{fig:d-swatch-filtered}, we plot a subset of the \(D\) test results, restricted to characters with less skewing.

\begin{figure}
\includegraphics[width=1\linewidth]{fig/d-scatterplots} \caption{Scatterplot of $D$ scores against (a) the number of doculects with non-missing values for each character and (b) the skewing of the distribution of 1s and 0s for each character. $D$ clusters evenly around the mean regardless of the number of missing values. Variation in $D$ accelerates greatly among characters where all but 1 or a few doculects share the same value, but the results are overwhelmingly not significant.}\label{fig:d-scatterplots}
\end{figure}

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/d-density-filtered} 

}

\caption{Density of $D$ estimates for binary biphone characters where character values are skewed less than 97-to-3.}\label{fig:d-density-filtered}
\end{figure}

\begin{figure}

{\centering \includegraphics[width=0.75\linewidth]{fig/d-swatch-filtered} 

}

\caption{$D$ scores for binary biphone characters where character values are skewed less than 97-to-3. Red shades show a character's proximity to $D=0$---darker red indicates stronger phylogenetic signal. Blue shades show a character's proximity to $D=1$, where character values are distributed randomly. This heat grid shows the proximity of each individual character's $D$ value to $D=0$---its expectation if the character evolved along the phylogeny following a Brownian motion process (the vertical axis represents the first segment of the biphone, the horizontal axis represents the second segment).}\label{fig:d-swatch-filtered}
\end{figure}

\hypertarget{phy-sig-cont}{%
\section{Phylogenetic signal in continuous phonotactic data}\label{phy-sig-cont}}

We test whether a higher degree of phylogenetic signal is detectable in continuous-valued biphone characters. As in the previous test, we take every possible sequence of two segments, or biphones, in our sample of 111 Pama-Nyungan wordlists. In this case, however, rather than simply coding for the presence of a biphone in a language's lexicon, we consider the relative frequencies of transitions between the segments in that biphone across the language's lexicon. For each biphone character, we take two values: The Markov chain forward transition probability---that is, for a biphone \(xy\), the probability of \(x\) being followed by \(y\), normalised over all instances of \(x\). This captures, if only in a basic way, our awareness that words do not consist of strings of independent segments, but rather the probability of observing some segment is very much dependent on what came before it. Secondly, we take Markov chain backward transition probabilities---that is, for the biphone \(xy\), the probability of \(y\) being preceeded by \(x\), normalised over all instances of \(y\) in the lexicon. The frequency characters we extract come from wordlists. This is advantageous in that they are somewhat independent of word frequency effects since each word is counted only once, in contrast to frequencies extracted from language corpora. On the other hand, speakers show sensitivity to phoneme frequencies in language use (for example, when coining novel words) \autocites{coleman_stochastic_1997}{zuraw_patterned_2000}{ernestus_predicting_2003}{albright_rules_2003}{eddington_spanish_2004}{hayes_stochastic_2006}{gordon_phonological_2016} so word frequency will likely have some effect on phoneme and biphone frequency even in a wordlist. Investigation of phylogenetic signal in frequency characters extracted from corpora versus wordlists may be a possibility for future study.

We quantify phylogenetic signal by estimating \(K\) \autocite{blomberg_testing_2003} individually for each character, using the \emph{multiPhylosignal} function, in the \emph{picante} package \autocite{kembel_picante:_2010}, in \emph{R} statistical software. The \(K\) test has somewhat greater statistical power than the \(D\) test, enabling us to apply the test to characters with as few as 20 non-missing values. Calculation of \(K\) works with non-zero values only, so zero values (where the language contains both segments \(x\) and \(y\) but \(x\) is never followed by \(y\), or vice versa) are considered not applicable and removed from calculation. A total of 490 characters (245 biphone forward transition probabilities and 245 backward transition probabilities) meet the criterion of at least 20 languages with non-missing and non-zero values for testing. Subsequently, to evaluate whether the level of phylogenetic signal is significant for a given character, we conduct Blomberg, Garland and Ives' (2003) randomization procedure with 10,000 random permutations per character.

To summarise, this testing procedure proceeds as follows. For each biphone frequency character:
- If the character has at least 20 non-NA values, then:
- If the character has at least two unique frequency values (i.e.~not every character value is the same), then:
- Calculate \(K\), and:
- Conduct randomisation procedure to calculate \(p\) for \(H_0: K = 0\).

Mean \(K\) for all 490 characters is 0.54 (\(SD\) 0.21) (Figure \ref{fig:k-density}). This is comparable to certain physiological traits presented as examples of biological traits with a high degree of phylogenetic signal by \textcite{blomberg_testing_2003}, for example, \(K=0.55\) for log body mass of primates. Using the \textcite{blomberg_testing_2003} randomisation procedure, we find a statistically significant degree of phylogenetic signal for 354 of 490 characters (180 forward transition characters, 174 backward transition characters), or 72\% of the total dataset.

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-density} 

}

\caption{Density of $K$ scores for all frequency-based biphone character (frequencies of both forward and backward transitions between segments).}\label{fig:k-density}
\end{figure}

We consider whether phylogenetic signal is higher or lower in certain kinds of biphone characters. Figure \ref{fig:k-swatch} shows a matrix of \(K\) scores for forward transition characters, with rows and columns arranged by phonological natural class. No clear pattern stands out.

\begin{figure}

{\centering \includegraphics[width=0.75\linewidth]{fig/k-swatch} 

}

\caption{Phylogenetic signal for forward transition frequencies. This heat grid shows $K$ scores for biphone characters (forward transition frequencies only). Each square represents a biphone (where the first segment is listed on the vertical axis and second segment on the horizontal axis). Data points are taken from the frequencies of each biphone, $xy$, over the total frequency of segment $x$ in each language, and then phylogenetic signal $K$ is measured for each biphone. Darker red shades indicate a stronger degree of phylogenetic signa. As with the $D$ test, no clear pattern of high versus low $K$ scores stands out, although there is a high degree of phylogenetic signal in the dataset overall.}\label{fig:k-swatch}
\end{figure}

\hypertarget{phy-sig-cont-robustness}{%
\subsection{Robustness checks}\label{phy-sig-cont-robustness}}

Although \(K\) is intended to be a measure of phylogenetic signal that is independent of tree size and shape, tree size and shape can have some effect on results in practice \autocite{munkemuller_how_2012}. We wish to check that the Pama-Nyungan tree does not contain any unusual properties that could cause either the K statistic or the randomization procedure to perform unexpectedly. To do this, we allow simulated characters to evolve specifically along the Pama-Nyungan reference tree. We vary the model of evolution, between perfect Brownian motion along the entire tree and pure randomness generated directly at the tips of the tree, by mixing different strengths of Brownian phylogenetic signal and non-phylogenetic noise. 1000 traits are simulated at each percentage point interval for 100,000 total simulated traits (in other words, 1000 traits simulated with 100\% Brownian motion, then 1000 traits simulated with 99\% Brownian motion and 1\% randomness, and so on, until the traits evolve 100\% at random). Each simulated trait is then tested for statistical significance using the randomisation procedure described in Section \ref{phylo-sig}, with 1000 repetitions to determine a \(p\) value. In a robust testing scenario, \(K\) will scale appropriately between 0 and 1 according to the level of Brownian motion and random noise being simulated, and the randomisation procedure will distinguish between traits with and without a significant degree of phylogenetic signal with a satisfactory amount of Type I (false positive) and Type II (false negative) errors.

The results are plotted in Figure \ref{fig:k-simulation-plots}. The \(K\) statistic shows a considerable degree of variability but, in the absence of substantial random noise, centres slightly below \(K=1\) which suggests the statistic is behaving as expected (if not slightly conservatively) when phylogenetic signal is present. For characters whose simulated evolution is near-random, the baseline of \(K\) seems to be elevated a little by our particular reference tree, with non-phylogenetic simulated characters ranging from the expected \(K=0\) to around \(K=0.3\). This should be kept in mind when interpreting \(K\) scores across our results. As for the randomisation procedure, Figure \ref{fig:k-simulation-plots}(b) shows the percentage of simulated traits that were identified as having a significant degree of phylogenetic signal (\(p < 0.05\)) at a given level of Brownian motion mixed with random noise. Above around 65\% Brownian motion, there are no Type II errors. The ability to detect significant phylogenetic signal drops as the level of random noise increases beyond 35\%, though overall the test's sensitivity seems acceptable. At the opposite extreme, where characters are simulated completely at random (and, therefore, there is no phylogenetic signal to detect) the randomisation procedure falsely detects phylogenetic signal 5.2\% of the time, very close to the expected false discovery rate of 5\% (given the conventional threshold for statistical significance of \(\alpha = 0.05\)). On the basis of these simulations, we are satisfied that randomisation procedure is sufficiently robust, given the particular size and shape of the Pama-Nyungan reference phylogeny.

\begin{figure}

{\centering \includegraphics[width=0.9\linewidth]{fig/phylosim} 

}

\caption{Behaviour of the $K$ statistic and randomization procedure with the Pama-Nyungan reference phylogeny. Artificial characters are simulated evolving along the phylogeny with varying levels of non-Brownian noise. Where a pure Brownian motion process operates, $K$ averages around 1, as expected. Where there is no Brownian process at all (and therefore no phylogenetic signal) $K$ is elevated to around 0.2---likely an artefact of this particular tree size and shape.}\label{fig:k-simulation-plots}
\end{figure}

As a final check, we consider whether the \(K\) statistic might be affected by the quantity of missing or `not applicable' values for a given character. We inspect this visually by plotting, for all biphone characters, the relation between a biphone's \(K\) score and the number of language varieties with non-missing data points on which \(K\) was calculated (Figure \ref{fig:k-scatterplot}). When \(K\) is calculated on fewer than around 40 non-missing values, the statistic shows a wider degree of variability. In addition, phylogenetic signal is deemed statistically significant for fewer characters in this range, suggesting that the quantity of missing values is affecting the statistical power of the test. However, all \(K\) scores cluster centrally around the mean regardless of the number of languages with non-missing values, suggesting that the mean \(K\) we observe for the dataset overall is not significantly affected by missing data.

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-scatterplot} 

}

\caption{Estimates of $K$ for forward and backward transition frequency characters plotted against the number of doculects with non-missing values for each character. Although some statistical power is lost and variability increases among characters with the most missing values, $K$ scores cluster evenly around mean $K$ (0.52).}\label{fig:k-scatterplot}
\end{figure}

\hypertarget{fwd-vs-bkwd}{%
\subsection{Forward transitions versus backward transitions}\label{fwd-vs-bkwd}}

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-fwd-bkwd} 

}

\caption{Distribution of $K$ scores for forward transition frequencies versus backward transition frequencies. We find no significant difference between these character types.}\label{fig:k-fwd-vs-bkwd}
\end{figure}

We find no significant difference in the means of \(K\) for forward transition characters (mean \(K =\) 0.54) and backward transition characters (mean \(K =\) 0.53) (\(t=\) 0.44, \(df=\) 487.98, \(p=\) 0.661, 95\% CI {[}-0.03, 0.05{]}). The distributions of \(K\) scores for forward and backward transitions are plotted in Figure \ref{fig:k-fwd-vs-bkwd}, showing a high degree of overlap between the two.

\hypertarget{norm-characters}{%
\subsection{Normalisation of character values}\label{norm-characters}}

Visual inspection of the density plots for each character shows there is a tendency for character data to be negatively skewed (the weight of the distribution is left-of-centre), although this is not universally the case. To test whether the particular, heavy-tailed nature of the data has an effect on tests for phylogenetic signal, we apply Tukey's Ladder of Powers transformation to each character in the dataset and re-run both the \(K\) test and randomization procedure. This is a power transformation, which makes the data fit a normal distribution as closely as possible. It does this by finding the power transformation value, \(\lambda\), that maximises the \(W\) statistic of the Shapiro-Wilk test for normality for each character individually. For our purposes, this transformation is effectively a change in the evolutionary model: A Brownian motion process is still assumed---a character value may wander up or down with equal probability---but, in this model, character values shift up or down along a transformed scale.

Mean \(K\) for normalised character data is 0.55 (\(SD\) 0.1966404). Of 490 characters, 408 or 83\% (208 forward transitions, 200 backward transitions) contain phylogenetic signal significantly above the random expectation. There is no statistically significant difference between mean \(K\) for untransformed data (0.54) versus mean \(K\) for normalised data (\(t=\) 0.59, \(df=\) 973.66, \(p=\) 0.558, 95\% CI {[}-0.02, 0.03{]})---see Figure \ref{fig:orig-vs-nrmlzd}.

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-orig-norm} 

}

\caption{Distributions of $K$ for untransformed character values and their normalised counterparts. We find no significant difference between these distributions.}\label{fig:orig-vs-nrmlzd}
\end{figure}

\hypertarget{phy-sig-classes}{%
\section{Phylogenetic signal in natural-class-based characters}\label{phy-sig-classes}}

One limitation of analysing phylogenetic signal in biphone characters is the assumption that every biphone character is a statistically independent observation. In historical linguistic processes, however, phonological segments rarely behave independently. Rather, sound changes are applied to whole sound classes, thereby affecting any one of various cross-cutting sets of phonological segments (and, therefore, biphone characters we have used in this study).

To account for this non-independence and more faithfully model what we know about how phonotactic systems operate in a language, we extract forward and backward transition probabilities for sequences of phonological features. For the purposes of this experiment, word boundaries are counted as a class and vowels are reduced to a single `vowel' class. Three sets of characters are extracted: foward and backward transition probabilities between natural classes based on place of articulation (segments belonging to the following classes: word boundary, labial, dental, alveolar, retroflex, palatal, velar, glottal, vowel); forward and backward transition probabilities between natural classes based on major places of articulation, where coronal contrasts have been collapsed (word boundary, labial, apical, laminal, velar, vowel); and natural classes based on manner of articulation (word boundary, obstruent, nasal, vibrant, lateral, glide, rhotic glide, vowel). The choice of natural classes is based on well-established principles of organisation among segments in Australian languages \autocites{dixon_languages_1980}{hamilton_phonetic_1996}{baker_word_2014}{round_segment_2020}{round_phonotactics_2020}.

\begin{table}

\caption{\label{tab:k-natural-classes-summary}Summary of $K$ analysis for forward and backward transition frequencies between different natural classes. The two rightmost columns indicate the total number of characters analysed and the percentage of those characters with a significant degree of phylogenetic signal according to the randomization procedure.}
\centering
\begin{tabular}[t]{lrrr}
\toprule
Classes & Mean K & n characters & significant (\%)\\
\midrule
Place & 0.61 & 126 & 94\\
Major place & 0.62 & 96 & 74\\
Manner & 0.59 & 88 & 66\\
\bottomrule
\end{tabular}
\end{table}

Table \ref{tab:k-natural-classes-summary} presents mean \(K\) and the proportion of significant characters for each of these three natural class-based datasets. All show highly similar distributions (Figure \ref{fig:k-natural-classes}). There is no statistically significant difference in the means of \(K\) for the three feature types, according to a one-way ANOVA (\(F\)(, ) = 0.46, \(p\) = 0.6288925). An Anderson-Darling k-sample test, which tests the hypothesis that \(k\) independent samples come from a common, unspecified distribution (i.e., no prior assumption about normality) also finds no significant difference in the distributions of \(K\) scores for the three natural class-based datasets (\(AD=\) 2.15, \(T.AD=\) 0.14, \(p=\) 0.34).

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-natural-classes} 

}

\caption{Comparison of $K$ scores for transitions between different kinds of natural classes. The differences between all three distributions are not statistically significant.}\label{fig:k-natural-classes}
\end{figure}

\hypertarget{classes-vs-biphones}{%
\subsection{Natural class-based characters versus biphones}\label{classes-vs-biphones}}

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-biphones-classes} 

}

\caption{Distributions of $K$ scores for biphone characters, coding the relative frequencies of transitions between phonological segments, and natural class-based characters, coding the relative frequencies of transitions between natural classes of segments. Phylogenetic signal is higher overall in the natural class-based dataset than the biphone-based dataset}\label{fig:k-biphones-vs-features}
\end{figure}

We compare the degree of phylogenetic signal in the biphone data tested in Section \ref{phy-sig-cont} to the natural class-based data tested here (see Figure \ref{fig:k-biphones-vs-features}). Mean \(K\) for all natural class-based characters is \texttt{round(mean(k\_all\_classes\$K),\ 2)}, which is significantly higher than the mean \(K\) for biphone data, \texttt{round(biphone\_meanK,\ 2)} (\(t=\) 4.37, \(df=\) 618.89, \(p=\) 0, 95\% CI {[}0.04, 0.1{]}). The Kolmogorov-Smirnov test, which is more sensitive to the overall shape of the distribution than a t-test comparison of means, also finds a significant distinction between \(K\) for biphone characters and \(K\) for natural class-based characters (\(D=\) 0.2, \(p < 0.001\)).

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

Computational phylogenetic methods are increasingly commonplace in historical linguistics. However, there has been relatively less consideration of the range of data types that might profitably be used with computational phylogenetic methods, beyond traditional, manually-assembled sets of lexical cognate data. In this study, we have considered the potential utility of quantitative phonotactic data for historical linguistics, for the reasons that quantitative phonotactic data is (i) readily extractible from basic wordlists, and (ii) may show certain kinds of historical conservatism, where the historical signal in more traditional lexical data would be affected by borrowing and lexical innovation.

We extracted frequencies of transitions between phonological segments in scrubbed and comparably segmented wordlists representing 111 Pama-Nyungan language varieties. As points of comparison, we extracted two additional datasets: Firstly, a binarised version of the dataset, which simply records whether or not particular two-segment sequences, a.k.a. biphones, are present in a language's wordlist. This is to emulate, in a simple sense, the kind of information which is often recorded in the phonology section of published descriptive grammars. We also extracted frequencies for transitions between natural classes of sounds. This is to account (at least, to some partial degree) for the fact that phonological segments tend not to evolve independently but pattern into natural classes, thereby limiting the independence of biphone-based variables.

To test whether historical information is preserved in our phonotactic datasets, we tested for phylogenetic signal, that is, the degree to which variance in the data reflects the evolutionary history of the 111 language varieties. We took an independent phylogenetic tree, inferred by the second author using lexical cognate data, and assumed a simple Brownian motion model of evolution. Our first key finding is that a significant degree of phylogenetic signal is detected in all three datasets---binary, segment-based and sound class-based. Finding phylogenetic signal in the binary dataset is somewhat surprising, given previous descriptions of homogeneity in the phonotactics of Australian languages. Our second key finding is that phylogenetic signal is significantly stronger in the higher-definition, frequency-based datasets than it is in binary dataset. In turn, phylogenetic signal in the sound class-based dataset is significantly stronger than the segment-based dataset. We took a closer look at certain comparisons within datasets, namely, whether there is a difference between forward and backward transitions, different kinds of sound classes, or between original (heavy-tailed) data and transformed data (to more closely fit a normal distribution). In all three cases, no significant differences were found.

\hypertarget{overall-robustness}{%
\subsection{Overall robustness}\label{overall-robustness}}

One important assumption in this study is that the tree being used as a reference phylogeny is an accurate depiction of the phylogeny for the languages in question. In the absence of time travel, it is impossible to observe directly the true Pama-Nyungan phylogeny and thus surely satisfy this assumption. Instead, we must rely on best available data and methods to infer the phylogeny. As described in Section \ref{ref-phylogeny}, the Pama-Nyungan phylogeny we use in this study was inferred by the second author using Bayesian computational phylogenetic methods, which produce a posterior sample of possible trees. The specific reference tree we use in this study is a best possible summation of this posterior sample as determined by the \emph{maximum clade credibility} method. Naturally, there is a degree of uncertainty associated with the topology of the maximum clade credibility tree. It may be the case that a better representation of the true Pama-Nyungan phylogeny exists among the many slight permutations contained within the posteior tree sample.

To evaluate the robustness of our results against phylogenetic uncertainty, we repeat the \(K\) test for phylogenetic signal on a subset of sound-class-based characters and each of a subset of 100 trees selected from the posterior sample. The subset of characters includes forward and backward transitions between place features, plus forward and backward transitions for manner features. The trees selected are the 100 best trees in the posterior sample, based on the maximum clade credibility metric. In this way, we capture a degree of uncertainty in the topology and branch lengths of the Pama-Nyungan phylogeny, while restricting our attention to a subset of the most credible alternatives.

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-mcct-posterior} 

}

\caption{Comparison of $K$ statistics using a 100-tree posterior sample versus the maximum clade credibility tree alone.}\label{fig:tree-uncertainty}
\end{figure}

214 sound class characters in total are tested, giving 21,400 \(K\) statistics in total (each character multiplied by 100 trees). The mean of these \(K\) scores is 0.59, which compares to a mean \(K\) of 0.6 for the same characters applied only to the maximum clade credibility tree as per \ref{phy-sig-classes}. This difference is not statistically significant (\(t=\) -0.95, \(df=\) 217.17, \(p=\) 0.342, 95\% CI {[}-0.05, 0.02{]}). The distributions of these \(K\) scores are illustrated in Figure \ref{fig:tree-uncertainty}.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{fig/wordlist_subset_sizes} 

}

\caption{Wordlists ranked by size. (A) shows every second wordlist in the language sample. (B) shows the middle 50\% of wordlists. Each subset contains the same number of wordlists but discrepancy in their size is greatly reduce in (B).}\label{fig:wordlist-subset-sizes}
\end{figure}

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{fig/k-IQR-vs-Every2nd} 

}

\caption{Comparison of $K$ statistics using only wordlists falling within the 25th an 75th quantiles (middle 50\%) for wordlist size, versus a sample of every 2nd wordlist (when ranked in order of size).}\label{fig:wordlist-uncertainty}
\end{figure}

In Section \ref{wordlists} we mentioned that our wordlists vary significantly is size. The length of a wordlist can correlate with many other linguistic properties that it has. For instance, in our data, wordlist length explains XX\% of the variance in the lists' mean phonemic word length (log-log regression, PVAL), that is, longer lists tend to contain longer words (presumably since shorter lists are weighted towards more basic, shorter vocabulary items). The existence of correlations like this means that it is not possible simply to `counterbalance' the length of wordlists by sub-sampling or resampling their items so that the resampled lists all have the same length. For example, the resampled lists that derive from longer underlying lists would still contain longer words. Nevertheless, it would still desirable to know whether our results in Sections \ref{phy-sig-bin}--\ref{phy-sig-classes} are unduly influenced by the disparities in our wordlist lengths, by manipulating it in a controlled fashion. To do this, we extracted two different language sub-samples from our dataset. For the first, we ranked all wordlists by size and selected every second list, producing a sample half the original size but with the same disparity in lengths. For the second, we ranked all wordlists by size and selected the middle 50\% of the ranking, again producing a sample of half the size but this time with heavily reduced disparity, as shown in Figure \ref{fig:wordlist-subset-sizes}. Wordlists in the middle 50\% range in length from 528 to 1361 (mean 872).

For these samples, we ran \(K\) tests on place and manner natural class characters. Our reasoning is that if wordlist disparity strongly affects the estimation of phylogenetic signal, then we should see a clear difference in the results. Mean \(K\) for the middle 50\% of wordlists is 0.77, which is somewhat greater than the mean for every second wordlist 0.7. This difference is statistically significant (\(t=\) 3.29, \(df=\) 421.21, \(p=\) 0.001, 95\% CI {[}0.03, 0.12{]}). This suggests that disparities in wordlist length are somewhat degrading the phylogenetic signal in our data, although there remains broad similarity between them, as pictured in Figure \ref{fig:wordlist-uncertainty}.

One attributing factor for this small degradation in phylogenetic signal may be measurement error among the bottom quartile of small wordlists. Throughout this study, we have assumed that all character values are accurate and do not account for measurement error. Accounting for measurement error when testing for phylogenetic signal is an area of active development in comparative biology \autocite{zheng_new_2009}. In future studies, this degradation in phylogenetic signal could be investigated by relating variation in \(K\) statistics carefully to various linguistic properties that correlate with the length of wordlists.

\hypertarget{limitations}{%
\subsection{Limitations}\label{limitations}}

Any investigation of phylogenetic signal in essence is an investigation of cross-linguistic (dis)similarity. Accordingly, whenever our representations of linguistic facts are altered, then the phylogenetic signal detectable in them will almost certainly change to some degree. In this paper our focus has been trained on the initial, fundamental question of whether phylogenetic signal is detectable in maximally simple phonotactic characters. However, as work like ours increases, one priority will be to investigate how investigators' choices about how data is represented affects results.

Relevant for the current study, in Section \ref{wordlists} we described a process of normalisation. The motivation for this was to attempt to minimize certain aspects of variation in phonological representation that can arise from variation in how different linguists analyses the same essential facts \autocites{chao_non-uniqueness_1934}{hockett_problem_1963}{hyman_universals_2008}{dresher_contrastive_2009}. While normalisation per se ought to improve the quality of cross-linguistic comparisons that the data enables, there is still the question of which targets one ought to normalise the data towards, on what effect that choice can have. For example, a reviewer asks whether our choice to split up complex segments might amplify phylogenetic signal if it leads to certain phylogenetically distributed complex segments counting instead as biphones. This can be answered in three ways. First, in the general case, since splitting segments changes representations, it will alter aspects of (dis)similarity in the data, and so is very likely to affect phylogenetic signal in some manner. Second, in this particular case, the reviewer is likely to be correct, due to details of our method. Any cross-linguistically rare, complex segment would likely get excluded from our dataset. This is because, although it would figure in certain biphones, we have made use only of biphones that reach a minimal level of recurrence across our language sample, and thus the biphones containing the rare segment quite likely would not qualify. However, if we split this complex segment into two segments, thus into a biphone, the resulting biphone may well have sufficient cross-linguistic recurrence to qualify for inclusion in our dataset, and subsequently may contribute to raising phylogenetic signal. A final observation on thi point is that such questions, about how choices in data representation interact with the results from corresponding quantitative analysis, are made tractable by our method of data preparation. Unlike many state-of-the-art cross-linguistic datasets, in which values for each language are hand-coded and thus incapable of being `recalculated' under altered assumptions, our phonotactic characters are generated algorithmically from an underlying, very rich dataset. With a change to algorithmic parameters, we can systematically split segments or glue them together, neutralise them or keep them distinct, and document what we have done and how. As mentioned, in this paper our focus is on the simple existence of phylogenetic signal. Our methods, though, naturally extend to enable comprehensive checking of such interactions between data choices and results. Ultimately, as a discipline, we would like this to be true for all typological research, not just phylogenetics \autocite{round_matthew_2017}. An advantage of our general approach, is that it open the doors to this rigorous mode of inquiry.

A further limitation of this study relates to the assumption that the data being tested for phylogenetic signal are independent of the data that was used to infer the reference phylogeny. In this study, the wordlists from which we extracted phonotactic characters contain, as a small subset, the basic vocabulary items from which lexical cognate characters were inferred and subsequently used to build the reference phylogeny. It is unclear exactly to what degree this inclusion of basic vocabulary compromises the independence of our reference tree and phonotactic data. A reviewer points out that cognate data and phonotactic data are still somewhat independent, even when extracted from identical wordlists, since phonotactic attributes are not directly called on to make cognacy judgements. Nevertheless, sound change affects both phonotactics and cognate identification, so some degree of non-indepenence is to be expected. To ascertain whether this effect is significant, future studies could parameterise the inclusion/exclusion of basic vocabulary from the phonotactic data.

One reviewer raises the correlation between phylogeny and geography. A noted limitation of phylogenetic comparative methods is the inability to account for geography as a possible confound \autocite{sookias_deep_2018} and this limitation applies to this study. Although we leave it as a priority for future work, the task of disentangling phylogeny and geography is not intractable. For example, \textcite{freckleton_space_2009} present a method for quantifying the relative degree of spatial versus phylogenetic effects in comparative characters.

One final point to note is that recent research suggests that phylogenetic signal can be inflated when character values evolve according to a Lévy process, where a character value can wander as per a Brownian motion process, but with the addition of discontinuous paths (i.e., sudden jumps in the character's value) \autocite{uyeda_rethinking_2018}. This is a realistic concern in the linguistic context, where segment frequencies are subject to sudden shifts caused by phonological mergers and splits. The possibility of Lévy-like evolutionary processes is a matter of concern also in comparative biology and methods to investigate it are subject to active development in that field \autocite{uyeda_rethinking_2018}.

\hypertarget{conclusion}{%
\section{Conclusion}\label{conclusion}}

Historical and synchronic comparative linguistics are increasingly making use of phylogenetic methods for the same reasons that led biologist to switch to them several decades ago. Our central contention has been that phylogenetic methods not only give us new ways of studying existing comparative data sets, but open up the possibility to derive insights from new kinds of data. Here we demonstrate the potential for phylogenetically investigating phonotactic data, by showing that it indeed contains the kind of phylogenetic signal which is the prerequisite for a whole spectrum of phylogenetic analyses.

We find significant phylogenetic signal for several hundred phonotactic characters extracted semi-automatically from 111 Pama-Nyungan wordlists, demonstrating that historical information is detectable in phonotactic data, even at the relatively simple level of biphones and despite ostensibly high phonological uniformity. Contrary to the prevailing view in literature on Australian languages, and contrary to the findings of an earlier pilot study on a much smaller language sample, we find that binary characters marking the presence or absence of biphones in a doculect contain enough phylogenetically-patterned variation to detect phylogenetic signal. However, we find that statistical power is relatively low when operating with coarse-grained binary data and quantification of the degree of phylogenetic signal is affected by a large number of low-variation characters, where all but one or a few doculects share the same value. We find stronger phylogenetic signal in biphone characters of forward and backward transition frequencies. This reaffirms the results of earlier work, for the first time on a sample of languages spanning an entire large family and the vast majority of a continent. It also reaffirms earlier findings that Australian phonologies show a greater level of variation than traditionally has been appreciated, once matters of frequency are taken into account. We find a significantly greater level of phylogenetic signal again in characters based on the frequencies of forward and backward transitions between natural sound classes. The sound-class-based approach reduces the quantity of characters available to test, but limits sparsity in the dataset and accounts somewhat better for the role of sound-classes in the evolutionary processes that affect phonotactic patterns in human language. Interestingly, although there exists considerable variation in the level of phylogenetic signal found in individual characters, we find no observable pattern to this variation in segment-based biphone characters nor between the mean levels of phylogenetic signal observed for different kinds of sound classes (e.g., characters concerning place versus manner of articulation).

This work has implications for comparative linguistics, both typological and historical. Firstly, we recommend the use of phylogenetic comparative methods in typological work where the phylogenetic independence of a language sample (or lack thereof) is paramount. In the immediate term, this should be the case for any typological work concerning phonotactics, even in parts of the world such as Australia where phonotactics traditionally have been assumed to be relatively independent of phylogeny. Beyond phonotactics, however, explicit measurements of phylogenetic signal can be made for any set of cross-linguistic data and this can be built into statistical analysis, even in the presence of gaps and uncertainty in phylogenetic knowledge. In two decades of quantitative development in historical linguistics, there has still been relatively limited consideration of the kinds of characters used for inferring linguistic histories. The phonotactic characters presented here can be extracted relatively simply and in large quantities from wordlists, even where a full descriptive grammar is not available. Here, we test only the degree to which patterns of variation in our data match our independent, pre-existing knowledge of the phylogenetic history of the Pama-Nyungan family, however, the results suggest that phonotactic data of this kind could be used where the phylogeny is less certain, either by incorporating phonotactic data into phylogenetic inference directly or by constraining parts of the tree where lexical data on its own returns some doubt.

\hypertarget{Acknowledgements}{%
\subsubsection{Acknowledgements}\label{Acknowledgements}}

JM-C is supported by an Australian Government Research Training Program Scholarship. ER is supported by Australian Research Council (ARC) Discovery Grant DE150101024 and a Language Evolution Grant from the ARC Centre of Excellence for the Dynamics of Language. CB is supported by National Science Foundation Grant NSF1423711. We gratefully acknowledge this support.

% ***************************************************
