%If you are presenting work which has been previously published, acknowledge this here.
% ***************************************************
% How to introduce a previously published chapter
% ***************************************************
%This is an example of how you might introduce a chapter that has been published previously. 
\cleartoevenpage
\pagestyle{empty}	
%Use this command (above) to suppress the header from the preceding chapter.

\noindent
Text from the following publication has been incorporated as Chapter~\ref{ch-pcms}:

\noindent
%\fullcite{DumyCitationKey}
Jayden L. Macklin-Cordes \& Erich R. Round. In prep. Phylogenetic comparative methods in linguistics. \emph{Details TBA}.

\begin{table}[h]
	\centering
	\begin{tabular}{clr}
		\toprule
		Contributor & Statement of contribution & \% \\
		\midrule
		\textbf{Jayden Macklin-Cordes}	& initial concept			& 100 \\
		                                & analysis     	            & 100  \\
		                                & theoretical derivations 	& 100  \\
		                                & writing of text 			& 100  \\
										& proof-reading				& 50  \\
		\midrule
		Erich Round						& supervision, guidance 	& 100 \\
										& proof-reading				& 50  \\
		\bottomrule
	\end{tabular}
\end{table}

The following chapter will form part of a publication which is in preparation. Macklin-Cordes has taken responsibility for the research essay component of this paper, which is what is presented here in this thesis. In the eventual published version, Round will contribute the results of a case study looking at laminal contrasts in Australian languages.

% ***************************************************
% Example of an internal chapter
% ***************************************************
%This is an internal chapter of the thesis.
%If you have a long title, you can supply an abbreviated version to print in the Table of Contents using the optional argument to the \chapter command.
\chapter[Phylogenetic Comparative Methods]{Phylogenetic Comparative Methods in Linguistics}
\label{ch-pcms}	%CREATE YOUR OWN LABEL.
\pagestyle{headings}

% ********* Enter your text below this line: ********

Historical linguistics is increasingly making use of phylogenetic methods. However, phylogeny is consequential for all comparative study, including synchronic typology. Many fields of science, including linguistic typology, have a long history of considering phylogenetic relationships in data sampling. However, a family of statistical methods, \emph{phylogenetic comparative methods}, enable the incorporation of phylogeny directly in a statistical model, with no loss of data. This paper clarifies the logic behind phylogenetic comparative methods, and argues for their applicability in linguistic typology. I make the case for phylogenetic comparative methods firstly by surveying responses to the issue of phylogenetic independence in linguistics and other fields of science. I find that all fields share, in origin, similar lines of development in sampling methodology to create phylogenetically independent samples. However, since embracing quantitative methods, comparative biologists have developed mathematical frameworks for identifying and modelling phylogenetic effects in statistical analysis. Linguists have the opportunity to incorporate this wealth of experience into comparative research design. Latter sections of the paper outline how this can be done, firstly by describing \emph{phylogenetic signal} and methods for its measurement. Secondly, I survey existing applications of phylogenetic signal methods and phylogenetic comparative methods more generally in linguistics. Drawing together previous sections, I argue for the continued uptake of phylogenetic comparative methods in linguistics and describe some implications of this. A desirable outcome for future research would be the collation of high-resolution phylogenetic language trees inferred over the past two decades, brought together into a common data structure. Furthermore, whole-family language sampling in concert with phylogenetic comparative methods should be considered in linguistic typology, beyond language samples that aim for much more sparse, global coverage.

\hypertarget{pcm-intro}{%
\section{Introduction}\label{pcm-intro}}

The comparative language sciences are indispensable in the study of human language and offer a unique contribution to the study of human history. Besides being an intriguing academic pursuit in its own right, historical linguistics can be triangulated with other fields, such as genetics, archaeology and anthropology, to infer early population movements and interactions between cultures \autocites[e.g.][]{hunley_genetic_2008}{gray_language_2009}{bouckaert_mapping_2012}{malaspinas_genomic_2016}{bouckaert_origin_2018}. Similarly, linguistic typology has been combined with fields including ecology and physiology to produce some remarkable theories on the evolution of human language \autocites[e.g.][]{everett_climate_2015}{everett_languages_2017}{bentz_evolution_2018}{blasi_human_2019}.

The fundamental task of comparison underlies both linguistic typology and historical linguistics. In this respect, they are synchronic and diachronic sides of the same coin. In the historical case, extant language data are compared to infer past relationships between languages that cannot be observed directly due to the passing of time. In typology, language data are compared as well, though with different aims. These aims principally concern the nature of human language itself---the limits of possibility and tendencies in language structures, for example. The task of cross-linguistic comparison is complicated, however, by the interwoven patterns of historical descent and language contact that inevitably bind languages to varying degrees and manifest in shared linguistic forms and features observable today. Consequently, shared histories must be taken into account before drawing any inferences from cross-linguistic datasets. The prevalence of a linguistic variable may give the appearance of a particular linguistic tendency if languages are considered independent, but it may also be explained by a single linguistic innovation which was subsequently inherited by a large number of descendant languages. Shared linguistic histories must also be considered in any statistical analysis, since independence of data points is a fundamental assumption of many statistical methods.

Non-independence due to shared histories through descent, termed \emph{phylogenetic non-independence} or, more precisely, \emph{phylogenetic autocorrelation}, is not an unfamiliar concept in linguistics, nor other fields where entities share common paths of descent, such as biology and anthropology. Over a century of thought and methodological development has been dedicated to the topic. However, divergences exist in the lines of thought and development of different fields. This paper considers this old discussion within a cross-disciplinary scope. There are many challenges associated with accounting for phylogenetic autocorrelation in comparative methods, however I find that certain challenges are not as unique to linguistics as often has been assumed. Comparative biologists continue to give a good deal of consideration to methodological challenges of interest to linguists as well. In particular, I find that a family of statistical methods, \emph{phylogenetic comparative methods} (PCMs), are immediately applicable to comparative linguistics. In this paper, I elucidate how and why this is the case and demonstrate empirically the need to account for phylogenetic autocorrelation in variables which might have been assumed to be distributed independently of phylogeny, based on previous descriptions---these variables concern the typology of laminal consonants in the Pama-Nyungan languages of Australia.

This paper proceeds as follows. Section \ref{phylo-autocorrelation} reviews literature on \emph{phylogenetic autocorrelation}---the tendency of languages to show similarities due to phylogenetic relatedness---in linguistics and cognate fields (comparative biology, in particular). The aim is to provide a broad picture of the scientific context that motivates the methodologies discussed later on. While identifying historical signal in a source of data is of clear interest to historical linguistics, it is also of interest to linguistic typology, where phylogenetic autocorrelation is a source of bias that must be controlled. I therefore survey literature from an array of comparative fields of science in which phylogenetic autocorrelation occurs, comparing methodological approaches for identifying and accounting for patterns of historical relatedness among observations in comparative datasets. Section \ref{phylo-sig} outlines some statistical tools for quantifying \emph{phylogenetic signal}, the degree of phylogenetic autocorrelation present in a comparative dataset. Quantifying phylogenetic signal is the first step in a phylogenetically informed comparative methodology---the presence or absence of phylogenetic signal determines the need for phylogenetic comparative methods in subsequent analysis. Then, in Section \ref{pcms-applications}, I discuss examples of studies measuring phylogenetic signal in linguistics. For instance, I consider the example of laminal phonemes in Pama-Nyungan languages (Australia). In this example, \textcite{round_continent-wide_2017} shows that, although place contrasts of laminal consonants are traditionally described as being areally distributed and independent of the Pama-Nyungan family's internal phylogeny, laminal consonants do show phylogenetic patterning when finer-grained variables are examined that capture matters of frequency. A finding like this emphasises the need to consider phylogenetic autocorrelation in linguistic typology, even when studying phenomena that might be considered `safe' from the effects of phylogeny based on prior research. I conclude by advocating for the continued uptake of phylogenetic comparative methods in linguistics. I discuss the implications of this uptake for language sampling strategy in linguistic typology and sketch an outline for future research.

\hypertarget{phylo-autocorrelation}{%
\section{Phylogenetic autocorrelation}\label{phylo-autocorrelation}}

Phylogenetic autocorrelation is common to many comparative fields of science, and linguistics is no exception. Phylogenetic autocorrelation is a potential problem for comparative study, because shared phylogenetic histories limit the independence of observations in a comparative dataset. Observations from more closely related entities will tend to show less variation than more distantly related entities, because they share a longer period of evolutionary history prior to splitting off from their most recent common ancestor, and will have had less time to diverge evolutionarily. If this tendency towards similarity due to shared phylogenetic history is not taken into account, it will introduce bias into the dataset and consequently affect statistical analysis. Different fields have their own lines of literature grappling with this phenomenon extending back many decades. Although there are many similarities between fields, key differences emerge since the uptake of quantitative methods in comparative biology. This section discusses phylogenetic autocorrelation and the history of responses to it in different fields, focusing in particular on linguistics (Section \ref{phylo-auto-ling}) and biology (Section \ref{phylo-auto-bio}).

\hypertarget{phylo-auto-ling}{%
\subsection{Phylogenetic autocorrelation in linguistics and other fields}\label{phylo-auto-ling}}

Both historical linguistics and linguistic typology are comparative fields, in that they necessarily rely on cross-linguistic datasets and the task of making comparisons between different languages is inherent to both. The presence of historical signal in a dataset---where the set of values reflects something of the history of featured languages---will be of interest to any researcher working comparatively, whether they are directly interested in reconstructing the history of those languages (as in historical linguistics) or not (as in linguistic typology). In the case of typology, this is because historical signal represents a kind of statistical non-independence between languages. Statistical non-independence between languages due to shared history is no new revelation in linguistic typology, however there are many possible approaches to dealing with it and a sizeable body of literature on the topic.

We explore typological literature further below, but first I take up the topic of phylogenetic non-independence in more detail. In any cross-linguistic study, there will be some degree of statistical non-independence between languages. This is because languages do not evolve independently, but share various historical connections through time, resulting in shared lexical or grammatical material, whether through inheritance from a common ancestor or borrowing from neighboring languages. Historical relationships between languages are, therefore, an inescapable concern for linguistic typology. Likewise, typological comparison has a place in historical linguistics, as researchers have investigated historical questions by applying statistical methods to typological datasets. A seminal example of this is \textcite{nichols_linguistic_1992}, and more recent examples within linguistic phylogenetics include \textcite{dunn_structural_2005}, \textcite{dunn_structural_2008}, \textcite{rexova_cladistic_2006}, \textcite{reesink_explaining_2009} and \textcite{greenhill_evolutionary_2017}. Non-independence of languages due to historical relationships, and the consequences for cross-linguistic comparison, has direct and indirect implications for both fields.

As noted above, linguistics is far from the only field to face the challenge of phylogenetic non-independence. In comparative anthropology, this issue was noted as early as 1889 by Sir Francis Galton in the context of cross-cultural datasets, which lack independence due to shared histories of cultural innovation and exchange between societies \autocite[p.~15]{naroll_two_1961}. This phenomenon, known as \emph{Galton's Problem}, is now more precisely understood as a form of statistical autocorrelation (similarity between observations as a function of the time lag between them). The same phenomenon has been recognised in comparative biology too. A seminal study concerning comparative studies of phenotypes, \textcite{felsenstein_phylogenies_1985} demonstrates that data from species cannot be assumed to be independently drawn from the same distribution, because species are related to one another via a branching, hierarchical phylogeny, thus, statistical methods that assume independent, identically-distributed observations will inflate the significance of the test (discussed further in Section \ref{phylo-auto-bio} below). Linguists, it has been argued, have been somewhat slower than those in other fields to acknowledge exposure to Galton's problem, or phylogenetic autocorrelation \autocite[p.~293]{perkins_statistical_1989}. Nevertheless, this is a central concern of \textcite[p.~259]{dryer_large_1989} and has been addressed in a considerable body of linguistic literature since then.

Although precise strategies are varied, common to all fields is a history of addressing phylogenetic autocorrelation at the data sampling stage. In linguistics, the use of sampling methods for creating a phylogenetically independent or phylogenetically balanced language sample remains the predominant way of accounting for phylogenetic autocorrelation and literature on this topic extends back several decades. \textcite[pp.~145--149]{bell_language_1978} argues that common strategies which simply ensure equally-weighted representation of ``all major families'' or all continents is inadequate due to differing rates of divergence among families. He estimates the number of language groups separated by more than 3,500 years of divergence and uses it as a heuristic for estimating genetic biases in a selection of proposed language samples. He concludes that European languages tended to be overrepresented and Indo-Pacific languages underrepresented and attributes this to a corresponding over/under-representation among quality language resources, which is a persistent problem for comparative linguistics. Perkins \autocites*{perkins_evolution_1980}{perkins_covariation_1988} creates a sample of 50 languages, later adapted by \textcite{bybee_morphology_1985}, which attempts to account for both genetic and areal biases by selecting no more than one language from each language phylum \autocite[following][]{voegelin_index_1966} and no more than one language from each cultural and geographic area \autocites[following][]{kenny_numerical_1975}{murdock_ethnographic_1967}. This method attempts to account for nonindependence due to areal spread, unlike Bell's heuristic measure which accounts only for genetic bias, however it does not account for differing ages of divergence and size of language phyla in the way Bell does. Crucially for this discussion, these sampling methods inherently have what I refer to as a \emph{lossy} quality. In information technology, lossy methods of data compression involve erasure of parts of the data to create a smaller approximation of the original file (for example, JPEG image files). Similarly, in the language sampling methods thus far described, a smaller approximation of a larger language sample is created by removing languages with certain historical connections that would compromise the independence of observations in the sample.

There are a number of shortfalls associated with lossy sampling strategies. At the crux of these is that it may not be possible to create an independent sample of sufficient size when distant genetic relationships and long-range areal phenomena are considered. There may be uncertainty and scope for disagreement on the independence of languages in a sample. \textcite[p.~261]{dryer_large_1989} refers to the example of the inclusion of three languages in Perkins' sample (Ingassana, Maasai and Songhai) which may be related as part of the Nilo-Saharan family, although these relationships are remote and subject to debate. When the largest proposed areal and genetic groupings are considered, it may simply not be possible to create an independent sample of a sufficient size for generating statistically significant inferences. Another problem is that the maximal extent of presently established language families is partially a product of the extent of adequate documentation and scholarly attention, rather than a completely true reflection of the fullest extent to which the family may be reconstructed. That is to say, two languages which are presently understood to be unrelated, and therefore statistically independent, may in fact belong to a shared larger grouping, which has not yet been identified due to poor documentation or some other factor. \textcite[p.~263]{dryer_large_1989} raises another concern, which is that languages selected on the basis of genetic independence may nonetheless share characteristics due to non-genetic processes---language contact and borrowing. When areal phenomena are considered, Dryer contends, the practicality of constructing a truly independent sample of sufficient size is further stretched. Dryer's proposed solution is to build a sample of languages of approximately equal relative independence (at the level of major subfamilies within Indo-European, such as Romance, Germanic, and so on) for each of five large linguistic areas which are assumed to be independent, or at least sufficiently independent for statistical purposes. Any statistical test can then be applied to each of the five areas and only if the same result is replicated in all five areas is it considered statistically significant. If the same result is replicated in four of five areas, this falls short of statistical significance, although \textcite[pp.~272--273]{dryer_large_1989} considers such cases to be evidence of a ``trend''. Even still, as \textcite[p.~284]{dryer_large_1989} acknowledges, his five linguistic areas may be subject to the same concerns about undetected historical non-independence and it is possible that the whole world may, in effect, function as a single linguistic area, such that the distribution of certain linguistic features may reflect extremely remote areal or genealogical pattern rather than some true tendency of human language.

\textcite[p.~41]{nichols_linguistic_1992} uses Dryer's area-by-area testing method as part of a three-pronged approach. For any given question, Nichols first conducts a chi-square test of the world sample and then re-tests the significance of the finding using either Dryer's method or by running the same test on only the sample of ``New World'' languages (comprising North, Central and South America). \textcite{rijkhoff_method_1993} and \textcite{rijkhoff_language_1998} develop another approach to account for the possibility of non-independence across large linguistic areas and large, as-yet-undetected families. They permit multiple languages within a family to be included but develop a measure, based on the density of nodes in a known language phylogeny, to determine how many languages should be included. In this way, they also aim to account for the fact that some language families will have greater internal diversity than others \autocites[see also][]{bakker_language_2011}{miestamo_sampling_2016}. Another proposed method is to set a minimum threshold of typological distance between languages, calculated from the \emph{World Atlas of Language Structures} (WALS) \autocite{dryer_wals_2013}, such that languages must be sufficiently typologically distinct from others in the sample to warrant inclusion. \textcite{bickel_refined_2009} develops an alternative algorithm based on \textcite{dryer_large_1989}, which allows all uniquely-valued data points within a family to be included in the sample, but then reduces the weighting of data points in the final analysis where a particular value is over-represented within a family. In other words, if all the languages in a particular family share the same value for a variable of interest, those observations may be reduced to a single data point, since this homogeneity is likely the result of a shared retention or innovation.

All up, the developments in typological methodology that have been discussed here demonstrate that historical non-independence between languages has been treated predominantly as a sampling issue in linguistics. Earlier researchers sought to maintain the independence of their sample by maximising the genetic distance between the languages in their sample, such that no two languages were known to belong to the same family. Later, with subsequent acknowledgement of the possibility of non-independence from very large language families, as well as large-scale areal diffusion and effects from as-yet undetected or unconfirmed historical relations, it became apparent that it may be impossible to create a sample which is simultaneously independent and sufficiently large to generate statistical significance. The response to this has been a variety of robustness checks, even bootstrapping-like processes, whereby languages are sampled at an approximately equal relative level of independence and the sample is then subdivided in some way and a statistical test replicated over each subdivision. More recent years have seen the continued evolution of statistics and robustness checking methods \autocite[for an overview, see][]{roberts_robust_2018}, although balanced sampling remains a common element of modern, large-scale comparative linguistic studies \autocites[for example,][]{everett_climate_2015}{everett_languages_2017}{blasi_grammars_2017}.

\hypertarget{phylo-auto-bio}{%
\subsection{Phylogenetic autocorrelation in comparative biology}\label{phylo-auto-bio}}

Comparative biology faces the same issue of phylogenetic autocorrelation as comparative linguistics. Many conventional statistical methods assume that observations are independent and identically distributed, which is problematic in biology since observations come from species, which are related to one another through shared evolutionary histories (as charted graphically in a tree diagram). Further, \textcite[p.~4]{felsenstein_phylogenies_1985} shows that even non-parametric statistics are not immune to violations of this independence assumption. Although both fields face the same phenomenon in essence, linguistics and biology have diverged in their methodological response in recent decades. While linguistics continues to focus on sampling procedures, giving less attention to the methods of subsequent statistical analysis, comparative biologists have shifted towards more direct, statistical solutions.

Earlier approaches to phylogenetic autocorrelation in biology are in a similar vein to the sampling methods discussed in the previous section. \textcite[pp.~346--347]{harvey_comparisons_1982} seek to find a taxonomic level to sample from, which strikes the right balance in terms of being sufficiently statistically independent without being so conservative that sample sizes become prohibitively small. Their proposed solution is to identify and sample from the lowest taxonomic level which can be ``justified on statistical grounds''. One method of doing this is suggested by \textcite[pp.~6--8]{clutton-brock_primate_1977}, who conduct a nested analysis of variance and then select taxonomic level containing the greatest level of variation. Once \textcite{clutton-brock_primate_1977} identify their taxonomic level of interest, they average out data for all species within a given genus for which they have data. In other words, the unit of analysis has shifted from individual species to genera, and each data point represents a genus in the form of an averaged representation of all the species within the genus. Although a similar method in essence, this genus-level averaging process is in marked contrast to balanced sampling methods discussed in the previous section, where an unaltered observation from a single exemplar language is taken as representative of its given family, subfamily or other defined grouping.

\textcite[pp.~85--86]{baker_evolution_1979} discuss the same problem. They use an approach not too dissimilar from the area-by-area robustness checking by \textcite{dryer_large_1989} and \textcite{nichols_linguistic_1992}. \textcite{baker_evolution_1979} replicate their analysis within individual families as well as within different ecological areas, with the assumption that if the same associations are observed within different groups as they are across the dataset as a whole, then one can discount the possibility that the full analysis is simply picking up differences between different families or different ecological groups. A contrasting approach, at least in instances where categorical data are of interest, is to reconstruct ancestral states throughout the phylogeny, enabling one to directly observe whether species which share a common trait do so because of (non-independent) shared inheritance from a common ancestor or whether the traits have evolved independently. \textcite{gittleman_phylogeny_1981} uses a \emph{parsimony model} to reconstruct states in this way. This is where the states of traits at ancestral nodes in a tree are reconstructed in such a way as to minimise the number of evolutionary changes that would need to take place to produce the observed data for extant species on the tips of the tree. There are a couple of shortfalls to this approach. One is that it is only a partial solution to the problem---it tells us which data points are independent and which are not, but besides potentially being used as a tool to identify a taxonomic level with the greatest level of diversity \autocite[proceeding in a similar way to][]{clutton-brock_primate_1977}, there is no indication of how to proceed with comparative study when non-independence has been identified. Further, there are biases in the parsimony method \autocite[p.~7]{felsenstein_phylogenies_1985} which are unlikely to be satisfying for linguists---for example, if a small group of related species (or languages) all share a trait, they will always be reconstructed as inheriting the shared trait from the nearest common ancestor, despite the possibility of parallel evolution (homoplasy), horizontal diffusion, environmental pressures, and so on.

\textcite{felsenstein_phylogenies_1985} contends that it is possible to account for phylogenetic non-independence in a statistical model without the need to remove non-independent data points or compromise the unit of analysis (by, for example, comparing averaged data points representing genera rather than individual species). Felsenstein's breakthrough insight is that this can be achieved not by directly comparing non-independent observations but by comparing \emph{phylogenetically independent contrasts} (PICs) between them. His method has become, by one estimate, the most widespread in comparative biology \autocite[p.~162]{nunn_comparative_2011}. Calculating phylogenetically independent contrasts is possible given a continuously-valued variable of interest, an assumed phylogeny and an assumed model of variable evolution. As a natural starting point, Felsenstein assumes a \emph{Brownian motion} model of evolution. This is where an evolving trait can wander positively or negatively with equal probability, and each new time step is independent from the last, with the resulting effect that displacement of the variable over time will be drawn from a normal distribution with a mean of zero and variance proportional to the amount of elapsed time \autocite[p.~8]{felsenstein_phylogenies_1985}. Consider two sister tips on a phylogenetic tree and two accompanying observations for a continuously-valued variable of interest. The two observations themselves cannot be considered statistically independent, since the two sister tips share much of their evolutionary history through a common point of origin. However, the \emph{contrast} between the two values is independent, because any difference between the two sisters will be the consequence of evolutionary events occurring only along the two \emph{separate} branches linking each sister to their last common ancestor. If the historical evolution of this variable follows a Brownian motion model as described above, then the contrast between the two sister tips will be drawn from a normal distribution with a mean of zero and variance proportional to the time that has elapsed since the two tips split in the tree. An observed contrast can be scaled by dividing it by the standard deviation of its expected variance. This gives a statistically independent contrast of expectation zero and unit variance. This process can be repeated for all adjacent tips in the tree. Contrasts can then be extracted from adjacent nodes in the tree, where the value of the node is an average of the observed values of the tips below it. In the end, there will be a collection of phylogenetic independent contrasts, all of expectation zero and unit variance. It is then possible to apply standard statistical tests to the phylogenetic independent contrasts (rather than directly to observed values) without phylogenetic autocorrelation introducing bias into the results.

One drawback of Felsenstein's method is the reliance on the assumption of Brownian motion as a model of variable evolution. \textcite{grafen_phylogenetic_1989} subsequently devises a similar method, \emph{the phylogenetic regression}, which has the flexibility to incorporate models of evolution other than Brownian motion. Further, Grafen's method is able to be applied in situations where phylogenetic information is incomplete (for example, where the phylogeny is an incomplete work-in-progress rather than an accepted gold-standard). This method is a phylogenetic adaptation of \emph{generalised least squares} (GLS). In this model, the value of a dependent variable, \(y_{i}\), is predicted by the equation \(y_{i} = \alpha + \beta x_{i} + \epsilon\), where \(\alpha\) is the intercept, \(\beta\) is the regression slope, \(x\) is the independent variable and \(\epsilon\) is an error term \autocite[p.~164]{nunn_comparative_2011}. Phylogenetic information can be incorporated into the error term, in the form of a variance-covariance matrix of phylogenetic distances between tips in a tree. PICs and GLS are mathematically equivalent when a Brownian motion evolutionary model is assumed and the reference tree is fully bifurcated, so PICs are essentially a special case of GLS where these assumptions are met \autocite{nunn_comparative_2011}.

There has been some interest in applying phylogenetic comparative methods to cross-linguistic data \autocites[for example,][]{dunn_evolved_2011}{maurits_tracing_2014}{verkerk_diachronic_2014}{birchall_comparison_2015}{zhou_quantifying_2015}{calude_typology_2016}{dunn_dative_2017}{verkerk_phylogenetic_2017}{bentz_evolution_2018}. In contrast to the lossy sampling methods discussed in the previous section, phylogenetic comparative methods are \emph{lossless}. They make it possible to incorporate all available data by directly incorporating phylogenetic history into the statistical model, rather than removing data points in order to balance the sample. Another reason that phylogenetic comparative methods are important is that no comparative study in either biology or linguistics is phylogenetically neutral, no matter what balanced sampling procedure might have been used. One may assume that a set of sampled languages are all sufficiently independent from one another and, accordingly, treat data from those languages as independent observations in statistical analysis, but this is still a phylogenetic assumption: one in which all languages are equally distant from one another in a phylogeny (in other words, all languages are connected to a single node by branches of the same length, with no intermediary structure---a \emph{star phylogeny}) \autocite{purvis_polytomies_1993}. Phylogenetic comparative methods enable the direct inclusion of existing phylogenetic knowledge beyond a simple star phylogeny.

One limitation of phylogenetic comparative methods, and a likely source of criticism in linguistics, is that they rely on access to high-quality, fully-resolved phylogenies complete with branch lengths. This simply is not realistic for families of languages across many parts of the world. Further, phylogenetic comparative methods assume reference phylogenies are accurate, when, in practice, phylogenies are subject to uncertainty. While tree inference in modern biology benefits from advances in high quality, large scale genomic data, it would be natural to balk at this assumption in the context of historical linguistics, where so many phylogenetic relationships within and between language families remain uncertain o unknown. It may come as somewhat of a surprise then, that \textcite[p.~14]{felsenstein_phylogenies_1985} expresses precisely the same concern about phylogenetic uncertainty in the context of comparative biology. Nevertheless, he says ``phylogenies are fundamental to comparative biology; there is no doing it without taking them into account'', and this is unavoidably true of comparative linguistics as well. Further, even if a study does not explicitly consider phylogeny, it is still unavoidably making phylogenetic assumptions. Comfortingly, computational advances make it feasible to incorporate phylogenetic uncertainty into analysis explicitly (and this is an area of continuing active development). Also, there is evidence that even when phylogenies are incomplete, lacking branch length information, or even subject to a degree of error, phylogenetic comparative methods still typically out-perform equivalent (non-phylogentic) comparative methods, which effectively assume a star phylogeny \autocites{grafen_phylogenetic_1989}{purvis_truth_1994}{symonds_effects_2002}.

\hypertarget{phylo-sig}{%
\section{Phylogenetic signal}\label{phylo-sig}}

As discussed in Section \ref{phylo-autocorrelation}, phylogenetic comparative methods are applicable in linguistic typology and any kind of comparative linguistic study where phylogeny is a confound. The previous section described phylogenetic comparative methods, which account for phylogeny. Some variables, however, may not evolve through descent with modification and may not pattern phylogenetically, or may do so only weakly. For example, some biological characteristics may reflect adaptations to a certain ecological niche. Ecological niche hypotheses have been proposed for some phonological variables too \autocites{everett_climate_2015}{everett_languages_2017}{blasi_grammars_2017}. How does one determine, then, whether phylogeny is a confounding factor for a variable of interest? In the last 15 years, an advance in this area has been the advent of methods for quantifying explicitly the degree of \emph{phylogenetic signal} in comparative data \autocites{freckleton_phylogenetic_2002}{blomberg_testing_2003}. Phylogenetic signal refers to the tendency of phylogenetically-related entities to resemble one another \autocites{blomberg_tempo_2002}[p.~717]{blomberg_testing_2003}. This resemblance is more technically defined as statistical non-independence among observation values due to phylogenetic relatedness between taxa \autocite[p.~591]{revell_phylogenetic_2008}. This concept of phylogenetic signal has important applications in comparative linguistics. Here I argue that measuring phylogenetic signal should be considered as a first step in a phylogenetically aware comparative methodology, since it can determine empirically whether phylogenetic comparative methods are required or whether regular statistical methods may suffice. Further, the result of a phylogenetic signal test can contribute to evolutionary hypotheses in its own right, for example by giving evidence for or against a variable following certain modes of evolution.

Rather than assuming phylogenetic non-independence \emph{a priori}, as the phylogenetic comparative methods discussed in Section \ref{phylo-auto-bio} do, or lacking any statistical control for phylogeny and relying on balance sampling alone, measures of phylogenetic signal provide the advantage of being able to quantify explicitly the degree of phylogenetic non-independence in a dataset \autocite[p.~591]{revell_phylogenetic_2008}. Phylogenetic signal may be expected to be strong in some cases or weak in others---given some data and a phylogenetic tree for reference, this can be tested empirically. If a typologist were to find a linguistic variable that is distributed significantly independently of phylogeny, this may be a result of interest in itself---at the least, it will provide an empirical basis that justifies proceeding with regular statistical methods over phylogenetic comparative methods \autocite[as in][]{irschick_comparison_1997}. Measures of phylogenetic signal will also be of interest to historical linguistics, as a diagnostic tool for testing the degree and nature in which some data reflect the phylogenetic history of the languages from which they came. The following section describes some methods for measuring phylogenetic signal in different kinds of variables.

\hypertarget{phylo-sig-quant}{%
\subsection{Quantifying phylogenetic signal in continuous variables}\label{phylo-sig-quant}}

\textcite{blomberg_testing_2003} provide a suite of tools for quantifying phylogenetic signal, which have become somewhat of a standard in the field (cited 3150 times on 31 May 2020, according to Google Scholar). Recent comparative studies using these tools include \textcite{balisi_dietary_2018}, \textcite{hutchinson_contemporary_2018} and \textcite{leff_predicting_2018}. \textcite{blomberg_testing_2003} present a descriptive statistic, \(K\), which is generalisable across phylogeneies of different sizes and shapes. In addition, they provide a randomisation test for checking whether the degree of phylogenetic signal for a given dataset is statistically significant. \(K\) can be calculated using either phylogenetic independent contrasts (PICs) \autocite{felsenstein_phylogenies_1985} or generalised least squares (GLS) \autocite{grafen_phylogenetic_1989} (see Section \ref{phylo-auto-bio}). In a Brownian motion model, where variable values can wander up and down with equal probability through time, PIC variances are expected to be proportional to elapsed time. Among more closely related languages, where there has been less divergence time for variable values to wander, the variance of PICs is expected to be low. The randomisation test works by comparing whether observed PICs are lower than the PIC values obtained by randomly permuting the data across the tips of the tree. The process of permuting data across tree tips at random is repeated many times over. If the real variances, with data in their correct positions on the tree, are lower than 95\% of the randomly permuted datasets, then the null hypothesis of no phylogenetic signal can be rejected at the conventional 95\% confidence level. In other words, closely related languages resemble one another to a statistically significantly greater degree than would be expected by chance.

The descriptive statistic, \(K\), quantifies the strength of phylogenetic signal. As with the randomisation procedure above, the input is a set of observed values, where each observation is associated with a tip of the reference tree. \textcite[p.~722]{blomberg_testing_2003} give an explanation of the calculation of the \(K\) statistic. To recap briefly, \(K\) is calculated by, firstly, taking the mean squared error of the data (\(MSE_0\)), as measured from a phylogenetically-corrected mean\footnote{Simply taking the mean of some variable would be misleading in cases where members of a particularly large clade happen to share similar values at an extreme end of the range. A phylogenetic mean is an estimate of the mean which has been corrected for overrepresentation by larger subclades \autocite[see][]{garland_polytomies_1999}.}, and dividing it by the mean squared error of the data (\(MSE\)), calculated using a variance-covariance matrix of phylogenetic distances between tips in the reference tree (the same variance-covariance matrix of phylogenetic distances incorporated into the error term in GLS-based phylogenetic regression, as discussed in the previous section). This latter value, \(MSE\), will be small when the pattern of covariance in the data matches what would be expected given the phylogenetic distances in the reference tree, leading to a high \(MSE_0/MSE\) ratio and vice versa. Thus, a high \(MSE_0/MSE\) ratio indicates higher phylogenetic signal. Finally, the observed ratio can be scaled according to its expectation under the assumption of Brownian motion evolution along the tree. This gives a \(K\) score which can be compared directly between analyses using different tree sizes and shapes. Where \(K = 1\), this suggests a perfect match between the covariance observed in the data and what would be expected given the reference tree and the assumption of Brownian motion evolution. Where \(K < 1\), close relatives in the tree bear less resemblance in the data than would be expected under the Brownian motion assumption. Notably, \(K > 1\) is also possible---this occurs where there is less variance in the data than expected, given the Brownian motion assumption and divergence times suggested by the reference tree. In other words, relatives bear greater resemblance than would be expected.

As discussed, the assumption of a Brownian motion model of evolution, where a variable is free to wander up or down, with equal probability, as time passes, is central to quantification of phylogenetic signal with the \(K\) statistic. \textcite[pp.~726--727]{blomberg_testing_2003} extend their approach to cover two different modes of evolution as well. This is achieved by incorporating extra parameters into the variance-covariance matrix to reflect different evolutionary processes. The first evolutionary model alternative is the Ornstein-Uhlenbeck (OU) model \autocites{felsenstein_phylogenies_1988}{garland_phylogenetic_1993}{hansen_translating_1996}{lavin_morphometrics_2008} whereby variables are still free to wander up or down at random, but there is a central pulling force towards some optimum value. The second alternative is an acceleration-deceleration (ACDC) model, developed by \textcite{blomberg_testing_2003} where a variable value moves up or down with equal probability (like Brownian motion) but the rate of evolution will either accelerate or decelerate over time.

Other statistics for quantifying phylogenetic signal have been proposed and warrant mention. \textcite{freckleton_phylogenetic_2002} propose using the \(\lambda\) (lambda) statistic, based on earlier work by \textcite{pagel_inferring_1999}. As for \textcite{blomberg_testing_2003}, this approach works with a variance-covariance matrix showing the amount of shared evolutionary history between any two tips in the tree (the diagonal of the matrix, the variances, will indicate the total height of the tree; the off-diagonals, the covariances, will indicate the amount of shared evolutionary history between two given entities, before they diverge in the tree). The statistic, \(\lambda\) is a scaling parameter which can be applied to this variance-covariance matrix. Scaling the values in the matrix by \(\lambda\) transforms the branch lengths of the tree, from \(\lambda = 1\), where branch lengths are left unscaled, to \(\lambda = 0\), where all covariances in the matrix will be zero, in other words, no covariance through shared evolutionary history is indicated between any tips, thus all tips will be joined at the root by branches of equal length (a star phylogeny). \textcite{freckleton_phylogenetic_2002} present a method for finding the \(\lambda\) parameter that maximises the likelihood of a set of observations arising, given a Brownian motion model of evolution. If \(\lambda\) is close to 1, this indicates high phylogenetic signal, where the data closely fit expectation given the shared evolutionary histories in the tree and a Brownian motion model of evolution. Further measures which have been proposed are \(I\) \autocite{moran_notes_1950}, a spatial autocorrelation measure which was adapted for phylogenetic analyses by \textcite{gittleman_adaptation:_1990}, and \(C_{mean}\) {[}abouheif\_method\_1999{]}, which is a test for serial independence \autocite[for an overview, see][]{munkemuller_how_2012}. In an evaluation of different methods \textcite{munkemuller_how_2012} find that, assuming a Brownian motion model of evolution, \(C_{mean}\) and \(\lambda\) generally outperform \(K\) and \(I\). However, \(C_{mean}\) considers only the topology of the reference tree (i.e., the order of the branches from top to bottom), but not branch length information, and the value of the \(C_{mean}\) statistic is partially dependent on tree size and shape, so it lacks comparability between different studies. In addition, \(\lambda\) shows some unreliability with small sample sizes (trees with \textasciitilde{}20 tips).

\hypertarget{phylo-sig-bin}{%
\subsection{Quantifying phylogenetic signal in binary variables}\label{phylo-sig-bin}}

The methods so far described concern continuously-valued data. Other methods have been proposed for quantifying phylogenetic signal in binary and categorical variables too. \textcite{abouheif_method_1999} presents a simulation-based approach for testing whether discrete values along the tips of a phylogeny are distributed in a phylogenetically non-random way. Although this method is useful for testing whether the phylogenetic signal in a set of discretely-valued data is statistically significant, it does not provide a quantification of the level of phylogenetic signal which is comparable between different datasets. Although specific to binary data only, \textcite{fritz_selectivity_2010} present a statistic, \(D\), which quantifies the strength of phylogenetic signal for some binary variable.

The \(D\) statistic is based on the sum of differences between sister tips and sister clades, \(\Sigma d\). To recap, following \textcite{fritz_selectivity_2010}, differences between values at the tips of the tree are summed first (all tips will either share the same value, 0 or 1, with 0 difference; or one will be 0 and the other will be 1, for a difference of 0.5). Nodes immediately above the tips are valued as an average of the two tips below (either 0, 0.5 or 1) and the differences between sister nodes is summed. This process is repeated for all nodes in the tree, until a total sum of differences, \(\Sigma d\), is reached. At two extremes, data may be maximally clumped, such that all 1s are grouped together in the same clade in the tree and likewise for all 0s, or data may be maximally dispersed, such that no two sister tips share the same value (every pair of sisters contains a 1 and a 0, leading to a maximal sum of differences). Lying somewhere in between will be both a phylogenetically random distribution and a distribution that is clumped to a degree expected under a Brownian motion model of evolution. A distribution of sums of differences following a phylogenetically random pattern, \(\Sigma d_r\), is obtained by shuffling variable values among tree tips many times over. A distribution of sums of differences following a Brownian motion pattern, \(\Sigma d_b\) is obtained by simulating the evolution of a continuous trait along the tree, following a Brownian motion process, many times over. Resulting values at the tips above a threshold are converted to 1, values below the threshold are converted to 0. The threshold is set to whatever level is required to obtain the same proportion of 1s and 0s as observed in the real data. Finally, \(D\) is determined by scaling the observed sum of differences to the means of the two reference distributions (the expected sums of differences under a phylogenetically random pattern and under a Brownian motion pattern).

\begin{equation}
D = \frac{\Sigma d_{obs} - mean\left( \Sigma d_{b} \right)}{mean\left( \Sigma d_{r} \right) - mean\left( \Sigma d_{b} \right)}
\end{equation}

Scaling \(D\) in this way provides a standardised statistic which can be compared between different sets of data, with trees of different sizes and shapes, as with \(K\) for continuous variables. One disadvantage of \(D\), however, is that it requires quite large sample sizes (\textgreater{}50), below which it loses statistical power, increasing the chance of a false positive result (type I error).

\hypertarget{phylo-sig-mult}{%
\subsection{Quantifying phylogenetic signal in multivariate and multidimensional data}\label{phylo-sig-mult}}

A notable, more recent development concerns the generalisation of methods for quantifying phylogenetic signal in multivariate and multi-dimensional data. Methods discussed so far quantify phylogenetic signal for a single variable of interest. \textcite{zheng_new_2009} present a generalisation of the \textcite{blomberg_testing_2003} \(K\) statistic for jointly estimating the strength of phylogenetic signal in a collection of variables. In addition, their method allows the incorporation of measurement error or variation within the entities being studied (be they species, languages, etcetera) \autocite[see][]{ives_within-species_2007}. Both of these developments are expected to improve the statistical power of the test, which is an advantage particularly where small sample sizes are concerned, since the original \(K\) statistic requires a minimum sample size of around 20 and lacks sufficient statistical power below this level. This is achieved by standardising the values of each variable to have mean 0 and variance 1 then jointly measuring the MSE for all variables. \textcite{adams_generalized_2014} presents another generalisation of \(K\) for use with `multivariate traits', \(K_{mult}\). Whereas \textcite{zheng_new_2009} estimate phylogenetic signal for a set of multiple \emph{independent} variables simultaneously, a multivariate trait is conceptually a single evolutionary trait but has multiple values associated with it, which are mathematically interrelated and cannot be analyzed independently. The example \textcite{adams_generalized_2014} uses is head shape for a family of salamander species.

\hypertarget{pcms-applications}{%
\section{Phylogenetic signal and PCMs in linguistics}\label{pcms-applications}}

\textcite{macklin-cordes_phylogenetic_2021} presents an application of phylogenetic signal measuring methods in linguistics, specifically a dataset of binary and continuous-valued phonotactic variables from 111 Pama-Nyungan languages using the \(D\) and \(K\) tests described above. Contrary to expectation, given prior descriptions of Pama-Nyungan phonology and phonotactics, strong phylogenetic signal is detected in this phonotactic data. This work has been tentatively expanded by \textcite{macklin-cordes_phylogeny_2018}. Since phonological processes and sound change affect natural classes of sounds rather than individual phonemes, phonotactic variables are subject to complex patterns of non-independence between them. \textcite{macklin-cordes_phylogeny_2018} attempts to account for this by implementing the \(K_{mult}\) method described above and shows that, at first pass, \(K_{mult}\) seems to detect stronger phylogenetic signal than testing several hundred phonotactic variables individually and averaging the results. This work requires further exploration, however. The evaluation of phylogenetic signal is, in itself, the end goal of these studies. They focus on the question of whether or not historical information is present in a novel dataset, which is more straightforwardly of interest for historical linguistic inquiry. However, the results hold secondary implications for the typological matter of non-independence between languages. The detection of phylogenetic signal in the phonotactics of Pama-Nyungan languages would need to be incorporated into any future typological methodology. The need for phylogenetic comparative methods has been established in this particular domain.

In another phylogenetic signal detection example using the same Pama-Nyungan reference phylogeny, \textcite{round_continent-wide_2017} re-examines assumptions about the laminal contrast in Australian languages. The distribution of Australian languages with a single contrastive laminal versus two contrastive laminal places at first seems not to correspond strongly with family and subgroup boundaries, leading researchers to propose that the distribution of one versus two laminal languages has been driven historically by areal diffusion \autocites{dixon_languages_1970}{dixon_languages_1980}{breen_taps_1997}{dixon_australian_2002}. \textcite{round_continent-wide_2017}, however, extracts a finer grained level of variation by considering not just the single, binary presence or absence of a laminal contrast but the frequencies at which those laminals appear in certain contexts before and after different vowels. \textcite{round_continent-wide_2017} detects strong phylogenetic signal in these frequency variables, suggesting that, contrary to previous proposals, laminals do not appear to evolve in a distinctly non-phylogenetic way.

Without delving into detail, other recent examples of phylogenetic comparative methodologies include \textcite{bromham_rate_2015}, which examines the relationship between rates of lexical change to population size in Austronesian languages; \textcite{verkerk_where_2015}, investigating the development of manner verbs and path verbs in Indo-European languages; \textcite{calude_typology_2016}, reconstructing numerals systems in Indo-European languages; \textcite{bentz_evolution_2018}, investigating links between language evolution and ecological factors in a sample of nearly 7,000 languages across the world; and \textcite{moran_investigating_2020}, quantifying differential rates of change in consonants versus vowels in 8 language families across 6 continents.

there are a few limitations of PCMs to note. One limiting assumption is that the tree being used as a reference phylogeny is an accurate representation of the true historical phylogeny. Since the past cannot be observed directly, the best available yardstick is limited to the best available phylogeny that has been inferred independently. In linguistics, this can be a particular problem. The cross-linguistic coverage of linguistic phylogenetic studies has expanded rapidly in the past two decades and some families, such as Indo-European and Austronesian, have been studied extensively. However, high-resolution phylogenetic trees do not exist for large swathes of the globe's linguistic diversity. In the context of Sahul, there is good coverage of the Pama-Nyungan family \autocites{bowern_computational_2012}{bouckaert_origin_2018} but, as yet, few if any studies computationally inferring phylogenies of non-Pama-Nyungan families nor any Papuan language families in New Guinea to the north. There are several large databases of world language classifications and efforts to make them easily available for phylogenetic comparative study \autocite{dediu_making_2018}, but these can lack resolution and branch length information. One partial solution is to incorporate phylogenetic uncertainty into any PCM analysis explicitly by replicating the analysis over a posterior sample of trees rather than a single summary tree, replicating the analysis over separate, competing reference trees, or, in the case of a limited tree structure with lots of polytomies, randomly simulating bifurcating tree structures to simulate uncertainty.

A second key assumption that can prove problematic is that the data being tested are assumed to be completely independent of the data that was used to infer the reference phylogeny. This can be tricky in comparative linguistic data, which tend to contain complex interdependencies. As one example, the phonotactic frequency datasets in \textcite{macklin-cordes_phylogenetic_2021} were extracted from wordlists which contained, as a subset, the words that were used to code cognate data, from which the reference tree was inferred.

One final point to note is that recent research suggests that phylogenetic signal can be inflated when variable values evolve according to a Lévy process, where a variable value can wander as per a Brownian motion process, but with the addition of discontinuous paths (i.e., sudden jumps in the variable's value) \autocite{uyeda_rethinking_2018}. This is particularly concerning for any frequency-based phonological data, which will be subject to sudden shifts caused by phonemic mergers, splits, and other regular sound changes. This is likewise a matter of concern in comparative biology and subject to active development in that field \autocite{uyeda_rethinking_2018}.

\hypertarget{pcms-conclusion}{%
\section{Conclusion}\label{pcms-conclusion}}

Historical and synchronic comparative linguistics are increasingly making use of phylogenetic methods for the same reasons that led biologist to switch to them several decades ago. The central contention of this research essay has been that phylogenetic methods not only give us new ways of studying existing comparative data sets, but open up the possibility to derive insights from new kinds of data.

% ***************************************************
